cd /tmp/DynaMoth_Defects4J_Lang_58;
export JAVA_TOOL_OPTIONS="-Dfile.encoding=UTF8 -Duser.language=en-US -Duser.country=US -Duser.language=en";
TZ="America/New_York"; export TZ;
export PATH="/home/wp/tool/jdk1.8.0_71/bin/:$PATH";
export JAVA_HOME="/home/wp/tool/jdk1.8.0_71/bin/";
time java -Xmx1g -Xms1g -cp /home/wp/RepairThemAll/script/../repair_tools/nopol.jar:/home/wp/tool/jdk1.8.0_71/bin//../lib/tools.jar fr.inria.lille.repair.Main \
	--mode repair \
	--type pre_then_cond \
	--bugid Lang-58 \
	--oracle angelic \
	--synthesis dynamoth \
	--flocal gzoltar \
	--json \
	--solver z3 \
	--solver-path /home/wp/RepairThemAll/script/../libs/z3/build/z3 \
	--complianceLevel 7 \
	--source src/java/ \
	--classpath "target/classes/:target/tests/:/tmp/DynaMoth_Defects4J_Lang_58/target/classes/:/tmp/DynaMoth_Defects4J_Lang_58/target/tests/:/home/wp/RepairThemAll/script/../benchmarks/defects4j/framework/projects/Lang/lib/asm.jar:/home/wp/RepairThemAll/script/../benchmarks/defects4j/framework/projects/Lang/lib/easymock.jar:/home/wp/RepairThemAll/script/../benchmarks/defects4j/framework/projects/Lang/lib/commons-io.jar:/home/wp/RepairThemAll/script/../benchmarks/defects4j/framework/projects/Lang/lib/cglib.jar:/home/wp/RepairThemAll/script/../benchmarks/defects4j/framework/projects/lib/junit-4.11.jar:/home/wp/RepairThemAll/script/../repair_tools/nopol.jar";
	echo "\n\nNode: `hostname`\n";
	echo "\n\nDate: `date`\n";
Picked up JAVA_TOOL_OPTIONS: -Dfile.encoding=UTF8 -Duser.language=en-US -Duser.country=US -Duser.language=en
23:45:08.578 [main] INFO  fr.inria.lille.repair.nopol.NoPol - Source files: [src/java]
23:45:08.579 [main] INFO  fr.inria.lille.repair.nopol.NoPol - Classpath: [file:/tmp/DynaMoth_Defects4J_Lang_58/target/classes/, file:/tmp/DynaMoth_Defects4J_Lang_58/target/tests/, file:/tmp/DynaMoth_Defects4J_Lang_58/target/classes/, file:/tmp/DynaMoth_Defects4J_Lang_58/target/tests/, file:/home/wp/RepairThemAll/script/../benchmarks/defects4j/framework/projects/Lang/lib/asm.jar, file:/home/wp/RepairThemAll/script/../benchmarks/defects4j/framework/projects/Lang/lib/easymock.jar, file:/home/wp/RepairThemAll/script/../benchmarks/defects4j/framework/projects/Lang/lib/commons-io.jar, file:/home/wp/RepairThemAll/script/../benchmarks/defects4j/framework/projects/Lang/lib/cglib.jar, file:/home/wp/RepairThemAll/script/../benchmarks/defects4j/framework/projects/lib/junit-4.11.jar, file:/home/wp/RepairThemAll/script/../repair_tools/nopol.jar]
23:45:08.579 [main] INFO  fr.inria.lille.repair.nopol.NoPol - Statement type: PRE_THEN_COND
23:45:08.580 [main] INFO  fr.inria.lille.repair.nopol.NoPol - Args: [org.apache.commons.lang.ArrayUtilsAddTest, org.apache.commons.lang.ArrayUtilsAddTest, org.apache.commons.lang.ArrayUtilsRemoveTest, org.apache.commons.lang.ArrayUtilsRemoveTest, org.apache.commons.lang.ArrayUtilsTest, org.apache.commons.lang.ArrayUtilsTest, org.apache.commons.lang.BitFieldTest, org.apache.commons.lang.BitFieldTest, org.apache.commons.lang.BooleanUtilsTest, org.apache.commons.lang.BooleanUtilsTest, org.apache.commons.lang.CharEncodingTest, org.apache.commons.lang.CharEncodingTest, org.apache.commons.lang.CharRangeTest, org.apache.commons.lang.CharRangeTest, org.apache.commons.lang.CharSetTest, org.apache.commons.lang.CharSetTest, org.apache.commons.lang.CharSetUtilsTest, org.apache.commons.lang.CharSetUtilsTest, org.apache.commons.lang.CharUtilsTest, org.apache.commons.lang.CharUtilsTest, org.apache.commons.lang.ClassUtilsTest, org.apache.commons.lang.ClassUtilsTest, org.apache.commons.lang.EntitiesPerformanceTest, org.apache.commons.lang.EntitiesPerformanceTest, org.apache.commons.lang.EntitiesTest, org.apache.commons.lang.EntitiesTest, org.apache.commons.lang.IllegalClassExceptionTest, org.apache.commons.lang.IllegalClassExceptionTest, org.apache.commons.lang.IncompleteArgumentExceptionTest, org.apache.commons.lang.IncompleteArgumentExceptionTest, org.apache.commons.lang.IntHashMapTest, org.apache.commons.lang.IntHashMapTest, org.apache.commons.lang.LocaleUtilsTest, org.apache.commons.lang.LocaleUtilsTest, org.apache.commons.lang.NotImplementedExceptionTest, org.apache.commons.lang.NotImplementedExceptionTest, org.apache.commons.lang.NullArgumentExceptionTest, org.apache.commons.lang.NullArgumentExceptionTest, org.apache.commons.lang.NumberRangeTest, org.apache.commons.lang.NumberRangeTest, org.apache.commons.lang.NumberUtilsTest, org.apache.commons.lang.NumberUtilsTest, org.apache.commons.lang.ObjectUtilsTest, org.apache.commons.lang.ObjectUtilsTest, org.apache.commons.lang.RandomStringUtilsTest, org.apache.commons.lang.RandomStringUtilsTest, org.apache.commons.lang.SerializationUtilsTest, org.apache.commons.lang.SerializationUtilsTest, org.apache.commons.lang.StringEscapeUtilsTest, org.apache.commons.lang.StringEscapeUtilsTest, org.apache.commons.lang.StringUtilsEqualsIndexOfTest, org.apache.commons.lang.StringUtilsEqualsIndexOfTest, org.apache.commons.lang.StringUtilsIsTest, org.apache.commons.lang.StringUtilsIsTest, org.apache.commons.lang.StringUtilsSubstringTest, org.apache.commons.lang.StringUtilsSubstringTest, org.apache.commons.lang.StringUtilsTest, org.apache.commons.lang.StringUtilsTest, org.apache.commons.lang.StringUtilsTrimEmptyTest, org.apache.commons.lang.StringUtilsTrimEmptyTest, org.apache.commons.lang.SystemUtilsTest, org.apache.commons.lang.SystemUtilsTest, org.apache.commons.lang.UnhandledExceptionTest, org.apache.commons.lang.UnhandledExceptionTest, org.apache.commons.lang.ValidateTest, org.apache.commons.lang.ValidateTest, org.apache.commons.lang.WordUtilsTest, org.apache.commons.lang.WordUtilsTest, org.apache.commons.lang.builder.CompareToBuilderTest, org.apache.commons.lang.builder.CompareToBuilderTest, org.apache.commons.lang.builder.DefaultToStringStyleTest, org.apache.commons.lang.builder.DefaultToStringStyleTest, org.apache.commons.lang.builder.EqualsBuilderTest, org.apache.commons.lang.builder.EqualsBuilderTest, org.apache.commons.lang.builder.HashCodeBuilderAndEqualsBuilderTest, org.apache.commons.lang.builder.HashCodeBuilderAndEqualsBuilderTest, org.apache.commons.lang.builder.HashCodeBuilderTest, org.apache.commons.lang.builder.HashCodeBuilderTest, org.apache.commons.lang.builder.MultiLineToStringStyleTest, org.apache.commons.lang.builder.MultiLineToStringStyleTest, org.apache.commons.lang.builder.NoFieldNamesToStringStyleTest, org.apache.commons.lang.builder.NoFieldNamesToStringStyleTest, org.apache.commons.lang.builder.ReflectionToStringBuilderExcludeTest, org.apache.commons.lang.builder.ReflectionToStringBuilderExcludeTest, org.apache.commons.lang.builder.ShortPrefixToStringStyleTest, org.apache.commons.lang.builder.ShortPrefixToStringStyleTest, org.apache.commons.lang.builder.SimpleToStringStyleTest, org.apache.commons.lang.builder.SimpleToStringStyleTest, org.apache.commons.lang.builder.StandardToStringStyleTest, org.apache.commons.lang.builder.StandardToStringStyleTest, org.apache.commons.lang.builder.ToStringBuilderTest, org.apache.commons.lang.builder.ToStringBuilderTest, org.apache.commons.lang.builder.ToStringStyleTest, org.apache.commons.lang.builder.ToStringStyleTest, org.apache.commons.lang.enum.EnumTest, org.apache.commons.lang.enum.EnumTest, org.apache.commons.lang.enum.EnumUtilsTest, org.apache.commons.lang.enum.EnumUtilsTest, org.apache.commons.lang.enum.ValuedEnumTest, org.apache.commons.lang.enum.ValuedEnumTest, org.apache.commons.lang.enums.EnumEqualsTest, org.apache.commons.lang.enums.EnumEqualsTest, org.apache.commons.lang.enums.EnumTest, org.apache.commons.lang.enums.EnumTest, org.apache.commons.lang.enums.EnumUtilsTest, org.apache.commons.lang.enums.EnumUtilsTest, org.apache.commons.lang.enums.ValuedEnumTest, org.apache.commons.lang.enums.ValuedEnumTest, org.apache.commons.lang.exception.ExceptionUtilsTestCase, org.apache.commons.lang.exception.ExceptionUtilsTestCase, org.apache.commons.lang.exception.NestableDelegateTestCase, org.apache.commons.lang.exception.NestableDelegateTestCase, org.apache.commons.lang.exception.NestableErrorTestCase, org.apache.commons.lang.exception.NestableErrorTestCase, org.apache.commons.lang.exception.NestableExceptionTestCase, org.apache.commons.lang.exception.NestableExceptionTestCase, org.apache.commons.lang.exception.NestableRuntimeExceptionTestCase, org.apache.commons.lang.exception.NestableRuntimeExceptionTestCase, org.apache.commons.lang.math.DoubleRangeTest, org.apache.commons.lang.math.DoubleRangeTest, org.apache.commons.lang.math.FloatRangeTest, org.apache.commons.lang.math.FloatRangeTest, org.apache.commons.lang.math.FractionTest, org.apache.commons.lang.math.FractionTest, org.apache.commons.lang.math.IntRangeTest, org.apache.commons.lang.math.IntRangeTest, org.apache.commons.lang.math.LongRangeTest, org.apache.commons.lang.math.LongRangeTest, org.apache.commons.lang.math.NumberRangeTest, org.apache.commons.lang.math.NumberRangeTest, org.apache.commons.lang.math.NumberUtilsTest, org.apache.commons.lang.math.NumberUtilsTest, org.apache.commons.lang.math.RandomUtilsTest, org.apache.commons.lang.math.RandomUtilsTest, org.apache.commons.lang.math.RangeTest, org.apache.commons.lang.math.RangeTest, org.apache.commons.lang.mutable.MutableBooleanTest, org.apache.commons.lang.mutable.MutableBooleanTest, org.apache.commons.lang.mutable.MutableByteTest, org.apache.commons.lang.mutable.MutableByteTest, org.apache.commons.lang.mutable.MutableDoubleTest, org.apache.commons.lang.mutable.MutableDoubleTest, org.apache.commons.lang.mutable.MutableFloatTest, org.apache.commons.lang.mutable.MutableFloatTest, org.apache.commons.lang.mutable.MutableIntTest, org.apache.commons.lang.mutable.MutableIntTest, org.apache.commons.lang.mutable.MutableLongTest, org.apache.commons.lang.mutable.MutableLongTest, org.apache.commons.lang.mutable.MutableObjectTest, org.apache.commons.lang.mutable.MutableObjectTest, org.apache.commons.lang.mutable.MutableShortTest, org.apache.commons.lang.mutable.MutableShortTest, org.apache.commons.lang.text.CompositeFormatTest, org.apache.commons.lang.text.CompositeFormatTest, org.apache.commons.lang.text.StrBuilderAppendInsertTest, org.apache.commons.lang.text.StrBuilderAppendInsertTest, org.apache.commons.lang.text.StrBuilderTest, org.apache.commons.lang.text.StrBuilderTest, org.apache.commons.lang.text.StrLookupTest, org.apache.commons.lang.text.StrLookupTest, org.apache.commons.lang.text.StrMatcherTest, org.apache.commons.lang.text.StrMatcherTest, org.apache.commons.lang.text.StrSubstitutorTest, org.apache.commons.lang.text.StrSubstitutorTest, org.apache.commons.lang.text.StrTokenizerTest, org.apache.commons.lang.text.StrTokenizerTest, org.apache.commons.lang.time.DateFormatUtilsTest, org.apache.commons.lang.time.DateFormatUtilsTest, org.apache.commons.lang.time.DateUtilsTest, org.apache.commons.lang.time.DateUtilsTest, org.apache.commons.lang.time.DurationFormatUtilsTest, org.apache.commons.lang.time.DurationFormatUtilsTest, org.apache.commons.lang.time.FastDateFormatTest, org.apache.commons.lang.time.FastDateFormatTest, org.apache.commons.lang.time.StopWatchTest, org.apache.commons.lang.time.StopWatchTest]
23:45:08.582 [main] INFO  fr.inria.lille.repair.nopol.NoPol - Config: Config{synthesisDepth=3, collectStaticMethods=true, collectStaticFields=false, collectLiterals=false, onlyOneSynthesisResult=true, sortExpressions=true, maxLineInvocationPerTest=250, timeoutMethodInvocation=2000, dataCollectionTimeoutInSecondForSynthesis=900, addWeight=0.19478, subWeight=0.04554, mulWeight=0.0102, divWeight=0.00613, andWeight=0.10597, orWeight=0.05708, eqWeight=0.22798, nEqWeight=0.0, lessEqWeight=0.0255, lessWeight=0.0947, methodCallWeight=0.1, fieldAccessWeight=0.08099, constantWeight=0.14232, variableWeight=0.05195, mode=REPAIR, type=PRE_THEN_COND, synthesis=DYNAMOTH, oracle=ANGELIC, solver=Z3, solverPath='/home/wp/RepairThemAll/script/../libs/z3/build/z3', projectSources=[src/java], projectClasspath='[Ljava.net.URL;@1b40d5f0', projectTests=[org.apache.commons.lang.ArrayUtilsAddTest, org.apache.commons.lang.ArrayUtilsAddTest, org.apache.commons.lang.ArrayUtilsRemoveTest, org.apache.commons.lang.ArrayUtilsRemoveTest, org.apache.commons.lang.ArrayUtilsTest, org.apache.commons.lang.ArrayUtilsTest, org.apache.commons.lang.BitFieldTest, org.apache.commons.lang.BitFieldTest, org.apache.commons.lang.BooleanUtilsTest, org.apache.commons.lang.BooleanUtilsTest, org.apache.commons.lang.CharEncodingTest, org.apache.commons.lang.CharEncodingTest, org.apache.commons.lang.CharRangeTest, org.apache.commons.lang.CharRangeTest, org.apache.commons.lang.CharSetTest, org.apache.commons.lang.CharSetTest, org.apache.commons.lang.CharSetUtilsTest, org.apache.commons.lang.CharSetUtilsTest, org.apache.commons.lang.CharUtilsTest, org.apache.commons.lang.CharUtilsTest, org.apache.commons.lang.ClassUtilsTest, org.apache.commons.lang.ClassUtilsTest, org.apache.commons.lang.EntitiesPerformanceTest, org.apache.commons.lang.EntitiesPerformanceTest, org.apache.commons.lang.EntitiesTest, org.apache.commons.lang.EntitiesTest, org.apache.commons.lang.IllegalClassExceptionTest, org.apache.commons.lang.IllegalClassExceptionTest, org.apache.commons.lang.IncompleteArgumentExceptionTest, org.apache.commons.lang.IncompleteArgumentExceptionTest, org.apache.commons.lang.IntHashMapTest, org.apache.commons.lang.IntHashMapTest, org.apache.commons.lang.LocaleUtilsTest, org.apache.commons.lang.LocaleUtilsTest, org.apache.commons.lang.NotImplementedExceptionTest, org.apache.commons.lang.NotImplementedExceptionTest, org.apache.commons.lang.NullArgumentExceptionTest, org.apache.commons.lang.NullArgumentExceptionTest, org.apache.commons.lang.NumberRangeTest, org.apache.commons.lang.NumberRangeTest, org.apache.commons.lang.NumberUtilsTest, org.apache.commons.lang.NumberUtilsTest, org.apache.commons.lang.ObjectUtilsTest, org.apache.commons.lang.ObjectUtilsTest, org.apache.commons.lang.RandomStringUtilsTest, org.apache.commons.lang.RandomStringUtilsTest, org.apache.commons.lang.SerializationUtilsTest, org.apache.commons.lang.SerializationUtilsTest, org.apache.commons.lang.StringEscapeUtilsTest, org.apache.commons.lang.StringEscapeUtilsTest, org.apache.commons.lang.StringUtilsEqualsIndexOfTest, org.apache.commons.lang.StringUtilsEqualsIndexOfTest, org.apache.commons.lang.StringUtilsIsTest, org.apache.commons.lang.StringUtilsIsTest, org.apache.commons.lang.StringUtilsSubstringTest, org.apache.commons.lang.StringUtilsSubstringTest, org.apache.commons.lang.StringUtilsTest, org.apache.commons.lang.StringUtilsTest, org.apache.commons.lang.StringUtilsTrimEmptyTest, org.apache.commons.lang.StringUtilsTrimEmptyTest, org.apache.commons.lang.SystemUtilsTest, org.apache.commons.lang.SystemUtilsTest, org.apache.commons.lang.UnhandledExceptionTest, org.apache.commons.lang.UnhandledExceptionTest, org.apache.commons.lang.ValidateTest, org.apache.commons.lang.ValidateTest, org.apache.commons.lang.WordUtilsTest, org.apache.commons.lang.WordUtilsTest, org.apache.commons.lang.builder.CompareToBuilderTest, org.apache.commons.lang.builder.CompareToBuilderTest, org.apache.commons.lang.builder.DefaultToStringStyleTest, org.apache.commons.lang.builder.DefaultToStringStyleTest, org.apache.commons.lang.builder.EqualsBuilderTest, org.apache.commons.lang.builder.EqualsBuilderTest, org.apache.commons.lang.builder.HashCodeBuilderAndEqualsBuilderTest, org.apache.commons.lang.builder.HashCodeBuilderAndEqualsBuilderTest, org.apache.commons.lang.builder.HashCodeBuilderTest, org.apache.commons.lang.builder.HashCodeBuilderTest, org.apache.commons.lang.builder.MultiLineToStringStyleTest, org.apache.commons.lang.builder.MultiLineToStringStyleTest, org.apache.commons.lang.builder.NoFieldNamesToStringStyleTest, org.apache.commons.lang.builder.NoFieldNamesToStringStyleTest, org.apache.commons.lang.builder.ReflectionToStringBuilderExcludeTest, org.apache.commons.lang.builder.ReflectionToStringBuilderExcludeTest, org.apache.commons.lang.builder.ShortPrefixToStringStyleTest, org.apache.commons.lang.builder.ShortPrefixToStringStyleTest, org.apache.commons.lang.builder.SimpleToStringStyleTest, org.apache.commons.lang.builder.SimpleToStringStyleTest, org.apache.commons.lang.builder.StandardToStringStyleTest, org.apache.commons.lang.builder.StandardToStringStyleTest, org.apache.commons.lang.builder.ToStringBuilderTest, org.apache.commons.lang.builder.ToStringBuilderTest, org.apache.commons.lang.builder.ToStringStyleTest, org.apache.commons.lang.builder.ToStringStyleTest, org.apache.commons.lang.enum.EnumTest, org.apache.commons.lang.enum.EnumTest, org.apache.commons.lang.enum.EnumUtilsTest, org.apache.commons.lang.enum.EnumUtilsTest, org.apache.commons.lang.enum.ValuedEnumTest, org.apache.commons.lang.enum.ValuedEnumTest, org.apache.commons.lang.enums.EnumEqualsTest, org.apache.commons.lang.enums.EnumEqualsTest, org.apache.commons.lang.enums.EnumTest, org.apache.commons.lang.enums.EnumTest, org.apache.commons.lang.enums.EnumUtilsTest, org.apache.commons.lang.enums.EnumUtilsTest, org.apache.commons.lang.enums.ValuedEnumTest, org.apache.commons.lang.enums.ValuedEnumTest, org.apache.commons.lang.exception.ExceptionUtilsTestCase, org.apache.commons.lang.exception.ExceptionUtilsTestCase, org.apache.commons.lang.exception.NestableDelegateTestCase, org.apache.commons.lang.exception.NestableDelegateTestCase, org.apache.commons.lang.exception.NestableErrorTestCase, org.apache.commons.lang.exception.NestableErrorTestCase, org.apache.commons.lang.exception.NestableExceptionTestCase, org.apache.commons.lang.exception.NestableExceptionTestCase, org.apache.commons.lang.exception.NestableRuntimeExceptionTestCase, org.apache.commons.lang.exception.NestableRuntimeExceptionTestCase, org.apache.commons.lang.math.DoubleRangeTest, org.apache.commons.lang.math.DoubleRangeTest, org.apache.commons.lang.math.FloatRangeTest, org.apache.commons.lang.math.FloatRangeTest, org.apache.commons.lang.math.FractionTest, org.apache.commons.lang.math.FractionTest, org.apache.commons.lang.math.IntRangeTest, org.apache.commons.lang.math.IntRangeTest, org.apache.commons.lang.math.LongRangeTest, org.apache.commons.lang.math.LongRangeTest, org.apache.commons.lang.math.NumberRangeTest, org.apache.commons.lang.math.NumberRangeTest, org.apache.commons.lang.math.NumberUtilsTest, org.apache.commons.lang.math.NumberUtilsTest, org.apache.commons.lang.math.RandomUtilsTest, org.apache.commons.lang.math.RandomUtilsTest, org.apache.commons.lang.math.RangeTest, org.apache.commons.lang.math.RangeTest, org.apache.commons.lang.mutable.MutableBooleanTest, org.apache.commons.lang.mutable.MutableBooleanTest, org.apache.commons.lang.mutable.MutableByteTest, org.apache.commons.lang.mutable.MutableByteTest, org.apache.commons.lang.mutable.MutableDoubleTest, org.apache.commons.lang.mutable.MutableDoubleTest, org.apache.commons.lang.mutable.MutableFloatTest, org.apache.commons.lang.mutable.MutableFloatTest, org.apache.commons.lang.mutable.MutableIntTest, org.apache.commons.lang.mutable.MutableIntTest, org.apache.commons.lang.mutable.MutableLongTest, org.apache.commons.lang.mutable.MutableLongTest, org.apache.commons.lang.mutable.MutableObjectTest, org.apache.commons.lang.mutable.MutableObjectTest, org.apache.commons.lang.mutable.MutableShortTest, org.apache.commons.lang.mutable.MutableShortTest, org.apache.commons.lang.text.CompositeFormatTest, org.apache.commons.lang.text.CompositeFormatTest, org.apache.commons.lang.text.StrBuilderAppendInsertTest, org.apache.commons.lang.text.StrBuilderAppendInsertTest, org.apache.commons.lang.text.StrBuilderTest, org.apache.commons.lang.text.StrBuilderTest, org.apache.commons.lang.text.StrLookupTest, org.apache.commons.lang.text.StrLookupTest, org.apache.commons.lang.text.StrMatcherTest, org.apache.commons.lang.text.StrMatcherTest, org.apache.commons.lang.text.StrSubstitutorTest, org.apache.commons.lang.text.StrSubstitutorTest, org.apache.commons.lang.text.StrTokenizerTest, org.apache.commons.lang.text.StrTokenizerTest, org.apache.commons.lang.time.DateFormatUtilsTest, org.apache.commons.lang.time.DateFormatUtilsTest, org.apache.commons.lang.time.DateUtilsTest, org.apache.commons.lang.time.DateUtilsTest, org.apache.commons.lang.time.DurationFormatUtilsTest, org.apache.commons.lang.time.DurationFormatUtilsTest, org.apache.commons.lang.time.FastDateFormatTest, org.apache.commons.lang.time.FastDateFormatTest, org.apache.commons.lang.time.StopWatchTest, org.apache.commons.lang.time.StopWatchTest], complianceLevel=7, outputFolder=., json=true}
23:45:08.587 [main] INFO  fr.inria.lille.repair.nopol.NoPol - Available processors (cores): 1
23:45:08.590 [main] INFO  fr.inria.lille.repair.nopol.NoPol - Free memory: 951 MB
23:45:08.590 [main] INFO  fr.inria.lille.repair.nopol.NoPol - Maximum memory: 989 MB
23:45:08.590 [main] INFO  fr.inria.lille.repair.nopol.NoPol - Total memory available to JVM: 989 MB
23:45:08.590 [main] INFO  fr.inria.lille.repair.nopol.NoPol - Java version: 1.8.0_71
23:45:08.591 [main] INFO  fr.inria.lille.repair.nopol.NoPol - JAVA_HOME: /home/wp/tool/jdk1.8.0_71/bin/
23:45:08.591 [main] INFO  fr.inria.lille.repair.nopol.NoPol - PATH: /home/wp/tool/jdk1.8.0_71/bin/:/home/wp/RepairThemAll/benchmarks/defects4j/framework/bin:/home/wp/tool/gradle-5.4.1/bin:/home/wp/tool/apache-maven-3.5.2/bin:/home/wp/tool/jdk1.7.0_80/bin:/home/wp/tool/apache-ant-1.9.14/bin:/home/wp/tool/jdk1.7.0_80/bin:/home/wp/tool/jdk1.7.0_80/jre/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games
After filtering ......
rootclassname is org.apache.commons.lang.math.NumberUtils
line number is 452
23:45:45.677 [pool-2-thread-1] DEBUG fr.inria.lille.repair.nopol.NoPol - statement #1
23:45:45.678 [pool-2-thread-1] DEBUG fr.inria.lille.repair.nopol.NoPol - Analysing SourceLocation org.apache.commons.lang.math.NumberUtils:452 which is executed by 3 tests
-1898474807
23:45:46.619 [pool-2-thread-1] DEBUG fr.inria.lille.repair.nopol.NoPol - looking with class fr.inria.lille.repair.nopol.spoon.smt.ConditionalAdder
23:45:49.629 [pool-2-thread-1] DEBUG fr.inria.lille.repair.nopol.NoPol - looking with class fr.inria.lille.repair.nopol.spoon.smt.ConditionalReplacer
java -cp /tmp/DynaMoth_Defects4J_Lang_58/target/classes:/tmp/DynaMoth_Defects4J_Lang_58/target/tests:/tmp/DynaMoth_Defects4J_Lang_58/target/classes:/tmp/DynaMoth_Defects4J_Lang_58/target/tests:/home/wp/RepairThemAll/script/../benchmarks/defects4j/framework/projects/Lang/lib/asm.jar:/home/wp/RepairThemAll/script/../benchmarks/defects4j/framework/projects/Lang/lib/easymock.jar:/home/wp/RepairThemAll/script/../benchmarks/defects4j/framework/projects/Lang/lib/commons-io.jar:/home/wp/RepairThemAll/script/../benchmarks/defects4j/framework/projects/Lang/lib/cglib.jar:/home/wp/RepairThemAll/script/../benchmarks/defects4j/framework/projects/lib/junit-4.11.jar:/home/wp/RepairThemAll/script/../repair_tools/nopol.jar:/home/wp/RepairThemAll/repair_tools/nopol.jar:/home/wp/tool/jdk1.8.0_71/lib/tools.jar: fr.inria.lille.repair.MethodTestRunner org.apache.commons.lang.math.NumberUtilsTest#testCreateNumber org.apache.commons.lang.math.NumberUtilsTest#testLang300 org.apache.commons.lang.math.NumberUtilsTest#testIsNumber 
23:45:51.813 [pool-7-thread-1] DEBUG f.i.l.r.s.DynamothCodeGenesisImpl - ClassPrepareEvent
23:45:51.984 [pool-7-thread-1] DEBUG f.i.l.r.s.c.spoon.ClassCollector - [class] java.lang.String
23:45:51.989 [pool-7-thread-1] DEBUG f.i.l.r.s.c.spoon.ClassCollector - [class] org.apache.commons.lang.StringUtils
23:45:51.993 [pool-7-thread-1] DEBUG f.i.l.r.s.c.spoon.ClassCollector - [class] java.lang.NumberFormatException
23:45:52.006 [pool-7-thread-1] DEBUG f.i.l.r.s.c.spoon.ClassCollector - [class] org.apache.commons.lang.math.NumberUtils
23:45:52.032 [pool-7-thread-1] DEBUG f.i.l.r.s.c.spoon.ClassCollector - [class] java.lang.Integer
23:45:52.250 [pool-7-thread-1] DEBUG f.i.l.r.s.c.spoon.ClassCollector - [class] java.lang.Character
23:45:52.279 [pool-7-thread-1] DEBUG f.i.l.r.s.c.spoon.ClassCollector - [class] java.lang.Float
23:45:52.290 [pool-7-thread-1] DEBUG f.i.l.r.s.c.spoon.ClassCollector - [class] java.lang.Double
23:45:53.434 [pool-7-thread-1] DEBUG f.i.l.r.s.DynamothCodeGenesisImpl - BreakpointEvent
23:45:53.445 [pool-7-thread-1] INFO  f.i.l.r.s.DynamothCodeGenesisImpl - [test] org.apache.commons.lang.math.NumberUtilsTest#testCreateNumber iteration 0
23:45:53.463 [pool-7-thread-1] DEBUG f.i.l.r.s.c.SpoonElementsCollector - [data] dec=null
23:45:53.465 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - Collect Level 1
23:45:53.493 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] str
23:45:53.494 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] lastChar
23:45:53.495 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] decPos
23:45:53.496 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] expPos
23:45:53.496 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] mant
23:45:53.497 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] dec
23:45:53.497 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] exp
23:45:53.497 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] numeric
23:45:53.498 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] allZeros
23:45:53.577 [pool-15-thread-1] ERROR f.i.l.r.s.c.DynamothDataCollector - Unable to invoke the method org.apache.commons.lang.math.NumberUtils.createLong(java.lang.String) ["12345L"]
com.sun.jdi.InvocationException: Exception occurred in target VM
	at com.sun.tools.jdi.InvokableTypeImpl.invokeMethod(InvokableTypeImpl.java:129) ~[tools.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:439) [nopol.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:1) [nopol.jar:na]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) [na:1.8.0_71]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_71]
23:45:53.606 [pool-16-thread-1] ERROR f.i.l.r.s.c.DynamothDataCollector - Unable to invoke the method org.apache.commons.lang.math.NumberUtils.createLong(java.lang.String) ["12345L"]
com.sun.jdi.InvocationException: Exception occurred in target VM
	at com.sun.tools.jdi.InvokableTypeImpl.invokeMethod(InvokableTypeImpl.java:129) ~[tools.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:439) [nopol.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:1) [nopol.jar:na]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) [na:1.8.0_71]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_71]
23:45:53.621 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] org.apache.commons.lang.math.NumberUtils.createLong((java.lang.String) numeric)
23:45:53.645 [pool-18-thread-1] ERROR f.i.l.r.s.c.DynamothDataCollector - Unable to invoke the method org.apache.commons.lang.math.NumberUtils.createDouble(java.lang.String) ["12345L"]
com.sun.jdi.InvocationException: Exception occurred in target VM
	at com.sun.tools.jdi.InvokableTypeImpl.invokeMethod(InvokableTypeImpl.java:129) ~[tools.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:439) [nopol.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:1) [nopol.jar:na]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) [na:1.8.0_71]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_71]
23:45:53.660 [pool-19-thread-1] ERROR f.i.l.r.s.c.DynamothDataCollector - Unable to invoke the method org.apache.commons.lang.math.NumberUtils.createDouble(java.lang.String) ["12345L"]
com.sun.jdi.InvocationException: Exception occurred in target VM
	at com.sun.tools.jdi.InvokableTypeImpl.invokeMethod(InvokableTypeImpl.java:129) ~[tools.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:439) [nopol.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:1) [nopol.jar:na]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) [na:1.8.0_71]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_71]
23:45:53.670 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] org.apache.commons.lang.math.NumberUtils.createDouble((java.lang.String) numeric)
23:45:53.728 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] org.apache.commons.lang.StringUtils.isBlank((java.lang.String) str)
23:45:53.741 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] org.apache.commons.lang.StringUtils.isBlank((java.lang.String) mant)
23:45:53.746 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] org.apache.commons.lang.StringUtils.isBlank((java.lang.String) numeric)
23:45:53.786 [pool-24-thread-1] ERROR f.i.l.r.s.c.DynamothDataCollector - Unable to invoke the method org.apache.commons.lang.math.NumberUtils.createFloat(java.lang.String) ["12345L"]
com.sun.jdi.InvocationException: Exception occurred in target VM
	at com.sun.tools.jdi.InvokableTypeImpl.invokeMethod(InvokableTypeImpl.java:129) ~[tools.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:439) [nopol.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:1) [nopol.jar:na]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) [na:1.8.0_71]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_71]
23:45:53.790 [pool-25-thread-1] ERROR f.i.l.r.s.c.DynamothDataCollector - Unable to invoke the method org.apache.commons.lang.math.NumberUtils.createFloat(java.lang.String) ["12345L"]
com.sun.jdi.InvocationException: Exception occurred in target VM
	at com.sun.tools.jdi.InvokableTypeImpl.invokeMethod(InvokableTypeImpl.java:129) ~[tools.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:439) [nopol.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:1) [nopol.jar:na]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) [na:1.8.0_71]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_71]
23:45:53.796 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] org.apache.commons.lang.math.NumberUtils.createFloat((java.lang.String) numeric)
23:45:53.815 [pool-27-thread-1] ERROR f.i.l.r.s.c.DynamothDataCollector - Unable to invoke the method org.apache.commons.lang.math.NumberUtils.createBigDecimal(java.lang.String) ["12345L"]
com.sun.jdi.InvocationException: Exception occurred in target VM
	at com.sun.tools.jdi.InvokableTypeImpl.invokeMethod(InvokableTypeImpl.java:129) ~[tools.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:439) [nopol.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:1) [nopol.jar:na]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) [na:1.8.0_71]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_71]
23:45:53.824 [pool-28-thread-1] ERROR f.i.l.r.s.c.DynamothDataCollector - Unable to invoke the method org.apache.commons.lang.math.NumberUtils.createBigDecimal(java.lang.String) ["12345L"]
com.sun.jdi.InvocationException: Exception occurred in target VM
	at com.sun.tools.jdi.InvokableTypeImpl.invokeMethod(InvokableTypeImpl.java:129) ~[tools.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:439) [nopol.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:1) [nopol.jar:na]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) [na:1.8.0_71]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_71]
23:45:53.831 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] org.apache.commons.lang.math.NumberUtils.createBigDecimal((java.lang.String) numeric)
23:45:53.845 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] org.apache.commons.lang.math.NumberUtils.isDigits((java.lang.String) str)
23:45:53.849 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] org.apache.commons.lang.math.NumberUtils.isDigits((java.lang.String) mant)
23:45:53.856 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] org.apache.commons.lang.math.NumberUtils.isDigits((java.lang.String) numeric)
23:45:53.869 [pool-33-thread-1] ERROR f.i.l.r.s.c.DynamothDataCollector - Unable to invoke the method org.apache.commons.lang.math.NumberUtils.createBigInteger(java.lang.String) ["12345L"]
com.sun.jdi.InvocationException: Exception occurred in target VM
	at com.sun.tools.jdi.InvokableTypeImpl.invokeMethod(InvokableTypeImpl.java:129) ~[tools.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:439) [nopol.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:1) [nopol.jar:na]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) [na:1.8.0_71]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_71]
23:45:53.889 [pool-34-thread-1] ERROR f.i.l.r.s.c.DynamothDataCollector - Unable to invoke the method org.apache.commons.lang.math.NumberUtils.createBigInteger(java.lang.String) ["12345L"]
com.sun.jdi.InvocationException: Exception occurred in target VM
	at com.sun.tools.jdi.InvokableTypeImpl.invokeMethod(InvokableTypeImpl.java:129) ~[tools.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:439) [nopol.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:1) [nopol.jar:na]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) [na:1.8.0_71]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_71]
23:45:53.894 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] org.apache.commons.lang.math.NumberUtils.createBigInteger((java.lang.String) numeric)
23:45:53.902 [pool-36-thread-1] ERROR f.i.l.r.s.c.DynamothDataCollector - Unable to invoke the method org.apache.commons.lang.math.NumberUtils.createInteger(java.lang.String) ["12345L"]
com.sun.jdi.InvocationException: Exception occurred in target VM
	at com.sun.tools.jdi.InvokableTypeImpl.invokeMethod(InvokableTypeImpl.java:129) ~[tools.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:439) [nopol.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:1) [nopol.jar:na]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) [na:1.8.0_71]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_71]
23:45:53.907 [pool-37-thread-1] ERROR f.i.l.r.s.c.DynamothDataCollector - Unable to invoke the method org.apache.commons.lang.math.NumberUtils.createInteger(java.lang.String) ["12345L"]
com.sun.jdi.InvocationException: Exception occurred in target VM
	at com.sun.tools.jdi.InvokableTypeImpl.invokeMethod(InvokableTypeImpl.java:129) ~[tools.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:439) [nopol.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:1) [nopol.jar:na]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) [na:1.8.0_71]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_71]
23:45:53.921 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] org.apache.commons.lang.math.NumberUtils.createInteger((java.lang.String) numeric)
23:45:53.930 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] str.length()
23:45:53.942 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] mant.length()
23:45:53.951 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] numeric.length()
23:45:54.002 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] org.apache.commons.lang.math.NumberUtils.createBigDecimal((java.lang.String) numeric).intValue()
23:45:54.018 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] org.apache.commons.lang.math.NumberUtils.createBigInteger((java.lang.String) numeric).bitLength()
23:45:54.019 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] org.apache.commons.lang.math.NumberUtils.createBigInteger((java.lang.String) numeric).intValue()
23:45:54.021 [pool-7-thread-1] DEBUG f.i.l.r.s.DynamothCodeGenesisImpl - BreakpointEvent
23:45:54.021 [pool-7-thread-1] INFO  f.i.l.r.s.DynamothCodeGenesisImpl - [test] org.apache.commons.lang.math.NumberUtilsTest#testCreateNumber iteration 1
23:45:54.022 [pool-7-thread-1] DEBUG f.i.l.r.s.c.SpoonElementsCollector - [data] dec=null
23:45:54.022 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - Collect Level 1
23:45:54.022 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] str
23:45:54.022 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] lastChar
23:45:54.022 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] decPos
23:45:54.023 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] expPos
23:45:54.023 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] mant
23:45:54.023 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] dec
23:45:54.023 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] exp
23:45:54.023 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] numeric
23:45:54.023 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] allZeros
23:45:54.043 [pool-45-thread-1] ERROR f.i.l.r.s.c.DynamothDataCollector - Unable to invoke the method org.apache.commons.lang.math.NumberUtils.createLong(java.lang.String) ["12345l"]
com.sun.jdi.InvocationException: Exception occurred in target VM
	at com.sun.tools.jdi.InvokableTypeImpl.invokeMethod(InvokableTypeImpl.java:129) ~[tools.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:439) [nopol.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:1) [nopol.jar:na]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) [na:1.8.0_71]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_71]
23:45:54.046 [pool-46-thread-1] ERROR f.i.l.r.s.c.DynamothDataCollector - Unable to invoke the method org.apache.commons.lang.math.NumberUtils.createLong(java.lang.String) ["12345l"]
com.sun.jdi.InvocationException: Exception occurred in target VM
	at com.sun.tools.jdi.InvokableTypeImpl.invokeMethod(InvokableTypeImpl.java:129) ~[tools.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:439) [nopol.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:1) [nopol.jar:na]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) [na:1.8.0_71]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_71]
23:45:54.051 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] org.apache.commons.lang.math.NumberUtils.createLong((java.lang.String) numeric)
23:45:54.062 [pool-48-thread-1] ERROR f.i.l.r.s.c.DynamothDataCollector - Unable to invoke the method org.apache.commons.lang.math.NumberUtils.createDouble(java.lang.String) ["12345l"]
com.sun.jdi.InvocationException: Exception occurred in target VM
	at com.sun.tools.jdi.InvokableTypeImpl.invokeMethod(InvokableTypeImpl.java:129) ~[tools.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:439) [nopol.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:1) [nopol.jar:na]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) [na:1.8.0_71]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_71]
23:45:54.070 [pool-49-thread-1] ERROR f.i.l.r.s.c.DynamothDataCollector - Unable to invoke the method org.apache.commons.lang.math.NumberUtils.createDouble(java.lang.String) ["12345l"]
com.sun.jdi.InvocationException: Exception occurred in target VM
	at com.sun.tools.jdi.InvokableTypeImpl.invokeMethod(InvokableTypeImpl.java:129) ~[tools.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:439) [nopol.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:1) [nopol.jar:na]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) [na:1.8.0_71]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_71]
23:45:54.074 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] org.apache.commons.lang.math.NumberUtils.createDouble((java.lang.String) numeric)
23:45:54.081 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] org.apache.commons.lang.StringUtils.isBlank((java.lang.String) str)
23:45:54.086 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] org.apache.commons.lang.StringUtils.isBlank((java.lang.String) mant)
23:45:54.098 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] org.apache.commons.lang.StringUtils.isBlank((java.lang.String) numeric)
23:45:54.108 [pool-54-thread-1] ERROR f.i.l.r.s.c.DynamothDataCollector - Unable to invoke the method org.apache.commons.lang.math.NumberUtils.createFloat(java.lang.String) ["12345l"]
com.sun.jdi.InvocationException: Exception occurred in target VM
	at com.sun.tools.jdi.InvokableTypeImpl.invokeMethod(InvokableTypeImpl.java:129) ~[tools.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:439) [nopol.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:1) [nopol.jar:na]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) [na:1.8.0_71]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_71]
23:45:54.121 [pool-55-thread-1] ERROR f.i.l.r.s.c.DynamothDataCollector - Unable to invoke the method org.apache.commons.lang.math.NumberUtils.createFloat(java.lang.String) ["12345l"]
com.sun.jdi.InvocationException: Exception occurred in target VM
	at com.sun.tools.jdi.InvokableTypeImpl.invokeMethod(InvokableTypeImpl.java:129) ~[tools.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:439) [nopol.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:1) [nopol.jar:na]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) [na:1.8.0_71]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_71]
23:45:54.123 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] org.apache.commons.lang.math.NumberUtils.createFloat((java.lang.String) numeric)
23:45:54.131 [pool-57-thread-1] ERROR f.i.l.r.s.c.DynamothDataCollector - Unable to invoke the method org.apache.commons.lang.math.NumberUtils.createBigDecimal(java.lang.String) ["12345l"]
com.sun.jdi.InvocationException: Exception occurred in target VM
	at com.sun.tools.jdi.InvokableTypeImpl.invokeMethod(InvokableTypeImpl.java:129) ~[tools.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:439) [nopol.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:1) [nopol.jar:na]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) [na:1.8.0_71]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_71]
23:45:54.137 [pool-58-thread-1] ERROR f.i.l.r.s.c.DynamothDataCollector - Unable to invoke the method org.apache.commons.lang.math.NumberUtils.createBigDecimal(java.lang.String) ["12345l"]
com.sun.jdi.InvocationException: Exception occurred in target VM
	at com.sun.tools.jdi.InvokableTypeImpl.invokeMethod(InvokableTypeImpl.java:129) ~[tools.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:439) [nopol.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:1) [nopol.jar:na]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) [na:1.8.0_71]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_71]
23:45:54.141 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] org.apache.commons.lang.math.NumberUtils.createBigDecimal((java.lang.String) numeric)
23:45:54.150 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] org.apache.commons.lang.math.NumberUtils.isDigits((java.lang.String) str)
23:45:54.157 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] org.apache.commons.lang.math.NumberUtils.isDigits((java.lang.String) mant)
23:45:54.160 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] org.apache.commons.lang.math.NumberUtils.isDigits((java.lang.String) numeric)
23:45:54.165 [pool-63-thread-1] ERROR f.i.l.r.s.c.DynamothDataCollector - Unable to invoke the method org.apache.commons.lang.math.NumberUtils.createBigInteger(java.lang.String) ["12345l"]
com.sun.jdi.InvocationException: Exception occurred in target VM
	at com.sun.tools.jdi.InvokableTypeImpl.invokeMethod(InvokableTypeImpl.java:129) ~[tools.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:439) [nopol.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:1) [nopol.jar:na]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) [na:1.8.0_71]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_71]
23:45:54.171 [pool-64-thread-1] ERROR f.i.l.r.s.c.DynamothDataCollector - Unable to invoke the method org.apache.commons.lang.math.NumberUtils.createBigInteger(java.lang.String) ["12345l"]
com.sun.jdi.InvocationException: Exception occurred in target VM
	at com.sun.tools.jdi.InvokableTypeImpl.invokeMethod(InvokableTypeImpl.java:129) ~[tools.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:439) [nopol.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:1) [nopol.jar:na]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) [na:1.8.0_71]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_71]
23:45:54.174 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] org.apache.commons.lang.math.NumberUtils.createBigInteger((java.lang.String) numeric)
23:45:54.182 [pool-66-thread-1] ERROR f.i.l.r.s.c.DynamothDataCollector - Unable to invoke the method org.apache.commons.lang.math.NumberUtils.createInteger(java.lang.String) ["12345l"]
com.sun.jdi.InvocationException: Exception occurred in target VM
	at com.sun.tools.jdi.InvokableTypeImpl.invokeMethod(InvokableTypeImpl.java:129) ~[tools.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:439) [nopol.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:1) [nopol.jar:na]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) [na:1.8.0_71]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_71]
23:45:54.184 [pool-67-thread-1] ERROR f.i.l.r.s.c.DynamothDataCollector - Unable to invoke the method org.apache.commons.lang.math.NumberUtils.createInteger(java.lang.String) ["12345l"]
com.sun.jdi.InvocationException: Exception occurred in target VM
	at com.sun.tools.jdi.InvokableTypeImpl.invokeMethod(InvokableTypeImpl.java:129) ~[tools.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:439) [nopol.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:1) [nopol.jar:na]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) [na:1.8.0_71]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_71]
23:45:54.188 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] org.apache.commons.lang.math.NumberUtils.createInteger((java.lang.String) numeric)
23:45:54.191 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] str.length()
23:45:54.199 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] mant.length()
23:45:54.202 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] numeric.length()
23:45:54.226 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] org.apache.commons.lang.math.NumberUtils.createBigDecimal((java.lang.String) numeric).intValue()
23:45:54.231 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] org.apache.commons.lang.math.NumberUtils.createBigInteger((java.lang.String) numeric).bitLength()
23:45:54.238 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] org.apache.commons.lang.math.NumberUtils.createBigInteger((java.lang.String) numeric).intValue()
23:45:54.243 [pool-7-thread-1] DEBUG f.i.l.r.s.DynamothCodeGenesisImpl - BreakpointEvent
23:45:54.243 [pool-7-thread-1] INFO  f.i.l.r.s.DynamothCodeGenesisImpl - [test] org.apache.commons.lang.math.NumberUtilsTest#testCreateNumber iteration 2
23:45:54.246 [pool-7-thread-1] DEBUG f.i.l.r.s.c.SpoonElementsCollector - [data] dec=null
23:45:54.246 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - Collect Level 1
23:45:54.247 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] str
23:45:54.248 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] lastChar
23:45:54.249 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] decPos
23:45:54.249 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] expPos
23:45:54.249 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] mant
23:45:54.249 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] dec
23:45:54.249 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] exp
23:45:54.249 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] numeric
23:45:54.250 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] allZeros
23:45:54.257 [pool-75-thread-1] ERROR f.i.l.r.s.c.DynamothDataCollector - Unable to invoke the method org.apache.commons.lang.math.NumberUtils.createLong(java.lang.String) ["12345678901234567890L"]
com.sun.jdi.InvocationException: Exception occurred in target VM
	at com.sun.tools.jdi.InvokableTypeImpl.invokeMethod(InvokableTypeImpl.java:129) ~[tools.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:439) [nopol.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:1) [nopol.jar:na]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) [na:1.8.0_71]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_71]
23:45:54.262 [pool-76-thread-1] ERROR f.i.l.r.s.c.DynamothDataCollector - Unable to invoke the method org.apache.commons.lang.math.NumberUtils.createLong(java.lang.String) ["12345678901234567890L"]
com.sun.jdi.InvocationException: Exception occurred in target VM
	at com.sun.tools.jdi.InvokableTypeImpl.invokeMethod(InvokableTypeImpl.java:129) ~[tools.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:439) [nopol.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:1) [nopol.jar:na]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) [na:1.8.0_71]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_71]
23:45:54.266 [pool-77-thread-1] ERROR f.i.l.r.s.c.DynamothDataCollector - Unable to invoke the method org.apache.commons.lang.math.NumberUtils.createLong(java.lang.String) ["12345678901234567890"]
com.sun.jdi.InvocationException: Exception occurred in target VM
	at com.sun.tools.jdi.InvokableTypeImpl.invokeMethod(InvokableTypeImpl.java:129) ~[tools.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:439) [nopol.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:1) [nopol.jar:na]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) [na:1.8.0_71]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_71]
23:45:54.274 [pool-78-thread-1] ERROR f.i.l.r.s.c.DynamothDataCollector - Unable to invoke the method org.apache.commons.lang.math.NumberUtils.createDouble(java.lang.String) ["12345678901234567890L"]
com.sun.jdi.InvocationException: Exception occurred in target VM
	at com.sun.tools.jdi.InvokableTypeImpl.invokeMethod(InvokableTypeImpl.java:129) ~[tools.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:439) [nopol.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:1) [nopol.jar:na]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) [na:1.8.0_71]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_71]
23:45:54.288 [pool-79-thread-1] ERROR f.i.l.r.s.c.DynamothDataCollector - Unable to invoke the method org.apache.commons.lang.math.NumberUtils.createDouble(java.lang.String) ["12345678901234567890L"]
com.sun.jdi.InvocationException: Exception occurred in target VM
	at com.sun.tools.jdi.InvokableTypeImpl.invokeMethod(InvokableTypeImpl.java:129) ~[tools.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:439) [nopol.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:1) [nopol.jar:na]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) [na:1.8.0_71]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_71]
23:45:54.294 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] org.apache.commons.lang.math.NumberUtils.createDouble((java.lang.String) numeric)
23:45:54.310 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] org.apache.commons.lang.StringUtils.isBlank((java.lang.String) str)
23:45:54.318 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] org.apache.commons.lang.StringUtils.isBlank((java.lang.String) mant)
23:45:54.320 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] org.apache.commons.lang.StringUtils.isBlank((java.lang.String) numeric)
23:45:54.338 [pool-84-thread-1] ERROR f.i.l.r.s.c.DynamothDataCollector - Unable to invoke the method org.apache.commons.lang.math.NumberUtils.createFloat(java.lang.String) ["12345678901234567890L"]
com.sun.jdi.InvocationException: Exception occurred in target VM
	at com.sun.tools.jdi.InvokableTypeImpl.invokeMethod(InvokableTypeImpl.java:129) ~[tools.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:439) [nopol.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:1) [nopol.jar:na]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) [na:1.8.0_71]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_71]
23:45:54.342 [pool-85-thread-1] ERROR f.i.l.r.s.c.DynamothDataCollector - Unable to invoke the method org.apache.commons.lang.math.NumberUtils.createFloat(java.lang.String) ["12345678901234567890L"]
com.sun.jdi.InvocationException: Exception occurred in target VM
	at com.sun.tools.jdi.InvokableTypeImpl.invokeMethod(InvokableTypeImpl.java:129) ~[tools.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:439) [nopol.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:1) [nopol.jar:na]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) [na:1.8.0_71]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_71]
23:45:54.350 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] org.apache.commons.lang.math.NumberUtils.createFloat((java.lang.String) numeric)
23:45:54.356 [pool-87-thread-1] ERROR f.i.l.r.s.c.DynamothDataCollector - Unable to invoke the method org.apache.commons.lang.math.NumberUtils.createBigDecimal(java.lang.String) ["12345678901234567890L"]
com.sun.jdi.InvocationException: Exception occurred in target VM
	at com.sun.tools.jdi.InvokableTypeImpl.invokeMethod(InvokableTypeImpl.java:129) ~[tools.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:439) [nopol.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:1) [nopol.jar:na]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) [na:1.8.0_71]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_71]
23:45:54.361 [pool-88-thread-1] ERROR f.i.l.r.s.c.DynamothDataCollector - Unable to invoke the method org.apache.commons.lang.math.NumberUtils.createBigDecimal(java.lang.String) ["12345678901234567890L"]
com.sun.jdi.InvocationException: Exception occurred in target VM
	at com.sun.tools.jdi.InvokableTypeImpl.invokeMethod(InvokableTypeImpl.java:129) ~[tools.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:439) [nopol.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:1) [nopol.jar:na]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) [na:1.8.0_71]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_71]
23:45:54.374 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] org.apache.commons.lang.math.NumberUtils.createBigDecimal((java.lang.String) numeric)
23:45:54.382 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] org.apache.commons.lang.math.NumberUtils.isDigits((java.lang.String) str)
23:45:54.391 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] org.apache.commons.lang.math.NumberUtils.isDigits((java.lang.String) mant)
23:45:54.421 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] org.apache.commons.lang.math.NumberUtils.isDigits((java.lang.String) numeric)
23:45:54.430 [pool-93-thread-1] ERROR f.i.l.r.s.c.DynamothDataCollector - Unable to invoke the method org.apache.commons.lang.math.NumberUtils.createBigInteger(java.lang.String) ["12345678901234567890L"]
com.sun.jdi.InvocationException: Exception occurred in target VM
	at com.sun.tools.jdi.InvokableTypeImpl.invokeMethod(InvokableTypeImpl.java:129) ~[tools.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:439) [nopol.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:1) [nopol.jar:na]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) [na:1.8.0_71]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_71]
23:45:54.434 [pool-94-thread-1] ERROR f.i.l.r.s.c.DynamothDataCollector - Unable to invoke the method org.apache.commons.lang.math.NumberUtils.createBigInteger(java.lang.String) ["12345678901234567890L"]
com.sun.jdi.InvocationException: Exception occurred in target VM
	at com.sun.tools.jdi.InvokableTypeImpl.invokeMethod(InvokableTypeImpl.java:129) ~[tools.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:439) [nopol.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:1) [nopol.jar:na]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) [na:1.8.0_71]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_71]
23:45:54.441 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] org.apache.commons.lang.math.NumberUtils.createBigInteger((java.lang.String) numeric)
23:45:54.454 [pool-96-thread-1] ERROR f.i.l.r.s.c.DynamothDataCollector - Unable to invoke the method org.apache.commons.lang.math.NumberUtils.createInteger(java.lang.String) ["12345678901234567890L"]
com.sun.jdi.InvocationException: Exception occurred in target VM
	at com.sun.tools.jdi.InvokableTypeImpl.invokeMethod(InvokableTypeImpl.java:129) ~[tools.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:439) [nopol.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:1) [nopol.jar:na]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) [na:1.8.0_71]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_71]
23:45:54.459 [pool-97-thread-1] ERROR f.i.l.r.s.c.DynamothDataCollector - Unable to invoke the method org.apache.commons.lang.math.NumberUtils.createInteger(java.lang.String) ["12345678901234567890L"]
com.sun.jdi.InvocationException: Exception occurred in target VM
	at com.sun.tools.jdi.InvokableTypeImpl.invokeMethod(InvokableTypeImpl.java:129) ~[tools.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:439) [nopol.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:1) [nopol.jar:na]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) [na:1.8.0_71]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_71]
23:45:54.462 [pool-98-thread-1] ERROR f.i.l.r.s.c.DynamothDataCollector - Unable to invoke the method org.apache.commons.lang.math.NumberUtils.createInteger(java.lang.String) ["12345678901234567890"]
com.sun.jdi.InvocationException: Exception occurred in target VM
	at com.sun.tools.jdi.InvokableTypeImpl.invokeMethod(InvokableTypeImpl.java:129) ~[tools.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:439) [nopol.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:1) [nopol.jar:na]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) [na:1.8.0_71]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_71]
23:45:54.477 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] str.length()
23:45:54.486 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] mant.length()
23:45:54.490 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] numeric.length()
23:45:54.502 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] org.apache.commons.lang.math.NumberUtils.createBigDecimal((java.lang.String) numeric).intValue()
23:45:54.513 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] org.apache.commons.lang.math.NumberUtils.createBigInteger((java.lang.String) numeric).bitLength()
23:45:54.515 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] org.apache.commons.lang.math.NumberUtils.createBigInteger((java.lang.String) numeric).intValue()
23:45:54.517 [pool-7-thread-1] DEBUG f.i.l.r.s.DynamothCodeGenesisImpl - BreakpointEvent
23:45:54.517 [pool-7-thread-1] INFO  f.i.l.r.s.DynamothCodeGenesisImpl - [test] org.apache.commons.lang.math.NumberUtilsTest#testCreateNumber iteration 3
23:45:54.518 [pool-7-thread-1] DEBUG f.i.l.r.s.c.SpoonElementsCollector - [data] dec=null
23:45:54.518 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - Collect Level 1
23:45:54.519 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] str
23:45:54.519 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] lastChar
23:45:54.519 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] decPos
23:45:54.519 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] expPos
23:45:54.519 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] mant
23:45:54.520 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] dec
23:45:54.520 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] exp
23:45:54.520 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] numeric
23:45:54.520 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] allZeros
23:45:54.531 [pool-105-thread-1] ERROR f.i.l.r.s.c.DynamothDataCollector - Unable to invoke the method org.apache.commons.lang.math.NumberUtils.createLong(java.lang.String) ["102147483647L"]
com.sun.jdi.InvocationException: Exception occurred in target VM
	at com.sun.tools.jdi.InvokableTypeImpl.invokeMethod(InvokableTypeImpl.java:129) ~[tools.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:439) [nopol.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:1) [nopol.jar:na]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) [na:1.8.0_71]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_71]
23:45:54.542 [pool-106-thread-1] ERROR f.i.l.r.s.c.DynamothDataCollector - Unable to invoke the method org.apache.commons.lang.math.NumberUtils.createLong(java.lang.String) ["102147483647L"]
com.sun.jdi.InvocationException: Exception occurred in target VM
	at com.sun.tools.jdi.InvokableTypeImpl.invokeMethod(InvokableTypeImpl.java:129) ~[tools.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:439) [nopol.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:1) [nopol.jar:na]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) [na:1.8.0_71]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_71]
23:45:54.549 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] org.apache.commons.lang.math.NumberUtils.createLong((java.lang.String) numeric)
23:45:54.552 [pool-108-thread-1] ERROR f.i.l.r.s.c.DynamothDataCollector - Unable to invoke the method org.apache.commons.lang.math.NumberUtils.createDouble(java.lang.String) ["102147483647L"]
com.sun.jdi.InvocationException: Exception occurred in target VM
	at com.sun.tools.jdi.InvokableTypeImpl.invokeMethod(InvokableTypeImpl.java:129) ~[tools.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:439) [nopol.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:1) [nopol.jar:na]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) [na:1.8.0_71]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_71]
23:45:54.554 [pool-109-thread-1] ERROR f.i.l.r.s.c.DynamothDataCollector - Unable to invoke the method org.apache.commons.lang.math.NumberUtils.createDouble(java.lang.String) ["102147483647L"]
com.sun.jdi.InvocationException: Exception occurred in target VM
	at com.sun.tools.jdi.InvokableTypeImpl.invokeMethod(InvokableTypeImpl.java:129) ~[tools.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:439) [nopol.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:1) [nopol.jar:na]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) [na:1.8.0_71]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_71]
23:45:54.560 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] org.apache.commons.lang.math.NumberUtils.createDouble((java.lang.String) numeric)
23:45:54.572 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] org.apache.commons.lang.StringUtils.isBlank((java.lang.String) str)
23:45:54.573 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] org.apache.commons.lang.StringUtils.isBlank((java.lang.String) mant)
23:45:54.578 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] org.apache.commons.lang.StringUtils.isBlank((java.lang.String) numeric)
23:45:54.591 [pool-114-thread-1] ERROR f.i.l.r.s.c.DynamothDataCollector - Unable to invoke the method org.apache.commons.lang.math.NumberUtils.createFloat(java.lang.String) ["102147483647L"]
com.sun.jdi.InvocationException: Exception occurred in target VM
	at com.sun.tools.jdi.InvokableTypeImpl.invokeMethod(InvokableTypeImpl.java:129) ~[tools.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:439) [nopol.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:1) [nopol.jar:na]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) [na:1.8.0_71]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_71]
23:45:54.592 [pool-115-thread-1] ERROR f.i.l.r.s.c.DynamothDataCollector - Unable to invoke the method org.apache.commons.lang.math.NumberUtils.createFloat(java.lang.String) ["102147483647L"]
com.sun.jdi.InvocationException: Exception occurred in target VM
	at com.sun.tools.jdi.InvokableTypeImpl.invokeMethod(InvokableTypeImpl.java:129) ~[tools.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:439) [nopol.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:1) [nopol.jar:na]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) [na:1.8.0_71]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_71]
23:45:54.598 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] org.apache.commons.lang.math.NumberUtils.createFloat((java.lang.String) numeric)
23:45:54.603 [pool-117-thread-1] ERROR f.i.l.r.s.c.DynamothDataCollector - Unable to invoke the method org.apache.commons.lang.math.NumberUtils.createBigDecimal(java.lang.String) ["102147483647L"]
com.sun.jdi.InvocationException: Exception occurred in target VM
	at com.sun.tools.jdi.InvokableTypeImpl.invokeMethod(InvokableTypeImpl.java:129) ~[tools.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:439) [nopol.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:1) [nopol.jar:na]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) [na:1.8.0_71]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_71]
23:45:54.610 [pool-118-thread-1] ERROR f.i.l.r.s.c.DynamothDataCollector - Unable to invoke the method org.apache.commons.lang.math.NumberUtils.createBigDecimal(java.lang.String) ["102147483647L"]
com.sun.jdi.InvocationException: Exception occurred in target VM
	at com.sun.tools.jdi.InvokableTypeImpl.invokeMethod(InvokableTypeImpl.java:129) ~[tools.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:439) [nopol.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:1) [nopol.jar:na]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) [na:1.8.0_71]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_71]
23:45:54.614 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] org.apache.commons.lang.math.NumberUtils.createBigDecimal((java.lang.String) numeric)
23:45:54.623 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] org.apache.commons.lang.math.NumberUtils.isDigits((java.lang.String) str)
23:45:54.624 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] org.apache.commons.lang.math.NumberUtils.isDigits((java.lang.String) mant)
23:45:54.629 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] org.apache.commons.lang.math.NumberUtils.isDigits((java.lang.String) numeric)
23:45:54.633 [pool-123-thread-1] ERROR f.i.l.r.s.c.DynamothDataCollector - Unable to invoke the method org.apache.commons.lang.math.NumberUtils.createBigInteger(java.lang.String) ["102147483647L"]
com.sun.jdi.InvocationException: Exception occurred in target VM
	at com.sun.tools.jdi.InvokableTypeImpl.invokeMethod(InvokableTypeImpl.java:129) ~[tools.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:439) [nopol.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:1) [nopol.jar:na]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) [na:1.8.0_71]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_71]
23:45:54.642 [pool-124-thread-1] ERROR f.i.l.r.s.c.DynamothDataCollector - Unable to invoke the method org.apache.commons.lang.math.NumberUtils.createBigInteger(java.lang.String) ["102147483647L"]
com.sun.jdi.InvocationException: Exception occurred in target VM
	at com.sun.tools.jdi.InvokableTypeImpl.invokeMethod(InvokableTypeImpl.java:129) ~[tools.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:439) [nopol.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:1) [nopol.jar:na]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) [na:1.8.0_71]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_71]
23:45:54.648 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] org.apache.commons.lang.math.NumberUtils.createBigInteger((java.lang.String) numeric)
23:45:54.652 [pool-126-thread-1] ERROR f.i.l.r.s.c.DynamothDataCollector - Unable to invoke the method org.apache.commons.lang.math.NumberUtils.createInteger(java.lang.String) ["102147483647L"]
com.sun.jdi.InvocationException: Exception occurred in target VM
	at com.sun.tools.jdi.InvokableTypeImpl.invokeMethod(InvokableTypeImpl.java:129) ~[tools.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:439) [nopol.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:1) [nopol.jar:na]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) [na:1.8.0_71]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_71]
23:45:54.661 [pool-127-thread-1] ERROR f.i.l.r.s.c.DynamothDataCollector - Unable to invoke the method org.apache.commons.lang.math.NumberUtils.createInteger(java.lang.String) ["102147483647L"]
com.sun.jdi.InvocationException: Exception occurred in target VM
	at com.sun.tools.jdi.InvokableTypeImpl.invokeMethod(InvokableTypeImpl.java:129) ~[tools.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:439) [nopol.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:1) [nopol.jar:na]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) [na:1.8.0_71]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_71]
23:45:54.665 [pool-128-thread-1] ERROR f.i.l.r.s.c.DynamothDataCollector - Unable to invoke the method org.apache.commons.lang.math.NumberUtils.createInteger(java.lang.String) ["102147483647"]
com.sun.jdi.InvocationException: Exception occurred in target VM
	at com.sun.tools.jdi.InvokableTypeImpl.invokeMethod(InvokableTypeImpl.java:129) ~[tools.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:439) [nopol.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:1) [nopol.jar:na]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) [na:1.8.0_71]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_71]
23:45:54.670 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] str.length()
23:45:54.681 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] mant.length()
23:45:54.685 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] numeric.length()
23:45:54.706 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] org.apache.commons.lang.math.NumberUtils.createBigDecimal((java.lang.String) numeric).intValue()
23:45:54.716 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] org.apache.commons.lang.math.NumberUtils.createBigInteger((java.lang.String) numeric).bitLength()
23:45:54.725 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] org.apache.commons.lang.math.NumberUtils.createBigInteger((java.lang.String) numeric).intValue()
23:45:54.814 [pool-7-thread-1] DEBUG f.i.l.r.s.DynamothCodeGenesisImpl - BreakpointEvent
23:45:54.815 [pool-7-thread-1] INFO  f.i.l.r.s.DynamothCodeGenesisImpl - [test] org.apache.commons.lang.math.NumberUtilsTest#testLang300 iteration 0
23:45:54.815 [pool-7-thread-1] DEBUG f.i.l.r.s.c.SpoonElementsCollector - [data] dec=null
23:45:54.815 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - Collect Level 1
23:45:54.815 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] str
23:45:54.816 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] lastChar
23:45:54.816 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] decPos
23:45:54.816 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] expPos
23:45:54.816 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] mant
23:45:54.816 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] dec
23:45:54.816 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] exp
23:45:54.816 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] numeric
23:45:54.816 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] allZeros
23:45:54.825 [pool-135-thread-1] ERROR f.i.l.r.s.c.DynamothDataCollector - Unable to invoke the method org.apache.commons.lang.math.NumberUtils.createLong(java.lang.String) ["-1l"]
com.sun.jdi.InvocationException: Exception occurred in target VM
	at com.sun.tools.jdi.InvokableTypeImpl.invokeMethod(InvokableTypeImpl.java:129) ~[tools.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:439) [nopol.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:1) [nopol.jar:na]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) [na:1.8.0_71]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_71]
23:45:54.832 [pool-136-thread-1] ERROR f.i.l.r.s.c.DynamothDataCollector - Unable to invoke the method org.apache.commons.lang.math.NumberUtils.createLong(java.lang.String) ["-1l"]
com.sun.jdi.InvocationException: Exception occurred in target VM
	at com.sun.tools.jdi.InvokableTypeImpl.invokeMethod(InvokableTypeImpl.java:129) ~[tools.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:439) [nopol.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:1) [nopol.jar:na]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) [na:1.8.0_71]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_71]
23:45:54.842 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] org.apache.commons.lang.math.NumberUtils.createLong((java.lang.String) numeric)
23:45:54.854 [pool-138-thread-1] ERROR f.i.l.r.s.c.DynamothDataCollector - Unable to invoke the method org.apache.commons.lang.math.NumberUtils.createDouble(java.lang.String) ["-1l"]
com.sun.jdi.InvocationException: Exception occurred in target VM
	at com.sun.tools.jdi.InvokableTypeImpl.invokeMethod(InvokableTypeImpl.java:129) ~[tools.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:439) [nopol.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:1) [nopol.jar:na]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) [na:1.8.0_71]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_71]
23:45:54.863 [pool-139-thread-1] ERROR f.i.l.r.s.c.DynamothDataCollector - Unable to invoke the method org.apache.commons.lang.math.NumberUtils.createDouble(java.lang.String) ["-1l"]
com.sun.jdi.InvocationException: Exception occurred in target VM
	at com.sun.tools.jdi.InvokableTypeImpl.invokeMethod(InvokableTypeImpl.java:129) ~[tools.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:439) [nopol.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:1) [nopol.jar:na]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) [na:1.8.0_71]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_71]
23:45:54.870 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] org.apache.commons.lang.math.NumberUtils.createDouble((java.lang.String) numeric)
23:45:54.879 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] org.apache.commons.lang.StringUtils.isBlank((java.lang.String) str)
23:45:54.887 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] org.apache.commons.lang.StringUtils.isBlank((java.lang.String) mant)
23:45:54.893 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] org.apache.commons.lang.StringUtils.isBlank((java.lang.String) numeric)
23:45:54.902 [pool-144-thread-1] ERROR f.i.l.r.s.c.DynamothDataCollector - Unable to invoke the method org.apache.commons.lang.math.NumberUtils.createFloat(java.lang.String) ["-1l"]
com.sun.jdi.InvocationException: Exception occurred in target VM
	at com.sun.tools.jdi.InvokableTypeImpl.invokeMethod(InvokableTypeImpl.java:129) ~[tools.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:439) [nopol.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:1) [nopol.jar:na]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) [na:1.8.0_71]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_71]
23:45:54.916 [pool-145-thread-1] ERROR f.i.l.r.s.c.DynamothDataCollector - Unable to invoke the method org.apache.commons.lang.math.NumberUtils.createFloat(java.lang.String) ["-1l"]
com.sun.jdi.InvocationException: Exception occurred in target VM
	at com.sun.tools.jdi.InvokableTypeImpl.invokeMethod(InvokableTypeImpl.java:129) ~[tools.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:439) [nopol.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:1) [nopol.jar:na]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) [na:1.8.0_71]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_71]
23:45:54.917 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] org.apache.commons.lang.math.NumberUtils.createFloat((java.lang.String) numeric)
23:45:54.926 [pool-147-thread-1] ERROR f.i.l.r.s.c.DynamothDataCollector - Unable to invoke the method org.apache.commons.lang.math.NumberUtils.createBigDecimal(java.lang.String) ["-1l"]
com.sun.jdi.InvocationException: Exception occurred in target VM
	at com.sun.tools.jdi.InvokableTypeImpl.invokeMethod(InvokableTypeImpl.java:129) ~[tools.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:439) [nopol.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:1) [nopol.jar:na]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) [na:1.8.0_71]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_71]
23:45:54.936 [pool-148-thread-1] ERROR f.i.l.r.s.c.DynamothDataCollector - Unable to invoke the method org.apache.commons.lang.math.NumberUtils.createBigDecimal(java.lang.String) ["-1l"]
com.sun.jdi.InvocationException: Exception occurred in target VM
	at com.sun.tools.jdi.InvokableTypeImpl.invokeMethod(InvokableTypeImpl.java:129) ~[tools.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:439) [nopol.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:1) [nopol.jar:na]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) [na:1.8.0_71]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_71]
23:45:54.942 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] org.apache.commons.lang.math.NumberUtils.createBigDecimal((java.lang.String) numeric)
23:45:54.953 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] org.apache.commons.lang.math.NumberUtils.isDigits((java.lang.String) str)
23:45:54.955 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] org.apache.commons.lang.math.NumberUtils.isDigits((java.lang.String) mant)
23:45:54.957 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] org.apache.commons.lang.math.NumberUtils.isDigits((java.lang.String) numeric)
23:45:54.971 [pool-153-thread-1] ERROR f.i.l.r.s.c.DynamothDataCollector - Unable to invoke the method org.apache.commons.lang.math.NumberUtils.createBigInteger(java.lang.String) ["-1l"]
com.sun.jdi.InvocationException: Exception occurred in target VM
	at com.sun.tools.jdi.InvokableTypeImpl.invokeMethod(InvokableTypeImpl.java:129) ~[tools.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:439) [nopol.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:1) [nopol.jar:na]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) [na:1.8.0_71]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_71]
23:45:54.982 [pool-154-thread-1] ERROR f.i.l.r.s.c.DynamothDataCollector - Unable to invoke the method org.apache.commons.lang.math.NumberUtils.createBigInteger(java.lang.String) ["-1l"]
com.sun.jdi.InvocationException: Exception occurred in target VM
	at com.sun.tools.jdi.InvokableTypeImpl.invokeMethod(InvokableTypeImpl.java:129) ~[tools.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:439) [nopol.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:1) [nopol.jar:na]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) [na:1.8.0_71]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_71]
23:45:54.988 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] org.apache.commons.lang.math.NumberUtils.createBigInteger((java.lang.String) numeric)
23:45:54.994 [pool-156-thread-1] ERROR f.i.l.r.s.c.DynamothDataCollector - Unable to invoke the method org.apache.commons.lang.math.NumberUtils.createInteger(java.lang.String) ["-1l"]
com.sun.jdi.InvocationException: Exception occurred in target VM
	at com.sun.tools.jdi.InvokableTypeImpl.invokeMethod(InvokableTypeImpl.java:129) ~[tools.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:439) [nopol.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:1) [nopol.jar:na]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) [na:1.8.0_71]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_71]
23:45:55.005 [pool-157-thread-1] ERROR f.i.l.r.s.c.DynamothDataCollector - Unable to invoke the method org.apache.commons.lang.math.NumberUtils.createInteger(java.lang.String) ["-1l"]
com.sun.jdi.InvocationException: Exception occurred in target VM
	at com.sun.tools.jdi.InvokableTypeImpl.invokeMethod(InvokableTypeImpl.java:129) ~[tools.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:439) [nopol.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:1) [nopol.jar:na]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) [na:1.8.0_71]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_71]
23:45:55.010 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] org.apache.commons.lang.math.NumberUtils.createInteger((java.lang.String) numeric)
23:45:55.013 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] str.length()
23:45:55.017 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] mant.length()
23:45:55.025 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] numeric.length()
23:45:55.037 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] org.apache.commons.lang.math.NumberUtils.createBigDecimal((java.lang.String) numeric).intValue()
23:45:55.057 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] org.apache.commons.lang.math.NumberUtils.createBigInteger((java.lang.String) numeric).bitLength()
23:45:55.058 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] org.apache.commons.lang.math.NumberUtils.createBigInteger((java.lang.String) numeric).intValue()
23:45:55.059 [pool-7-thread-1] DEBUG f.i.l.r.s.DynamothCodeGenesisImpl - BreakpointEvent
23:45:55.059 [pool-7-thread-1] INFO  f.i.l.r.s.DynamothCodeGenesisImpl - [test] org.apache.commons.lang.math.NumberUtilsTest#testLang300 iteration 1
23:45:55.060 [pool-7-thread-1] DEBUG f.i.l.r.s.c.SpoonElementsCollector - [data] dec=null
23:45:55.060 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - Collect Level 1
23:45:55.061 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] str
23:45:55.061 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] lastChar
23:45:55.062 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] decPos
23:45:55.062 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] expPos
23:45:55.062 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] mant
23:45:55.062 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] dec
23:45:55.062 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] exp
23:45:55.062 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] numeric
23:45:55.062 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] allZeros
23:45:55.069 [pool-165-thread-1] ERROR f.i.l.r.s.c.DynamothDataCollector - Unable to invoke the method org.apache.commons.lang.math.NumberUtils.createLong(java.lang.String) ["01l"]
com.sun.jdi.InvocationException: Exception occurred in target VM
	at com.sun.tools.jdi.InvokableTypeImpl.invokeMethod(InvokableTypeImpl.java:129) ~[tools.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:439) [nopol.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:1) [nopol.jar:na]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) [na:1.8.0_71]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_71]
23:45:55.086 [pool-166-thread-1] ERROR f.i.l.r.s.c.DynamothDataCollector - Unable to invoke the method org.apache.commons.lang.math.NumberUtils.createLong(java.lang.String) ["01l"]
com.sun.jdi.InvocationException: Exception occurred in target VM
	at com.sun.tools.jdi.InvokableTypeImpl.invokeMethod(InvokableTypeImpl.java:129) ~[tools.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:439) [nopol.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:1) [nopol.jar:na]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) [na:1.8.0_71]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_71]
23:45:55.090 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] org.apache.commons.lang.math.NumberUtils.createLong((java.lang.String) numeric)
23:45:55.095 [pool-168-thread-1] ERROR f.i.l.r.s.c.DynamothDataCollector - Unable to invoke the method org.apache.commons.lang.math.NumberUtils.createDouble(java.lang.String) ["01l"]
com.sun.jdi.InvocationException: Exception occurred in target VM
	at com.sun.tools.jdi.InvokableTypeImpl.invokeMethod(InvokableTypeImpl.java:129) ~[tools.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:439) [nopol.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:1) [nopol.jar:na]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) [na:1.8.0_71]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_71]
23:45:55.110 [pool-169-thread-1] ERROR f.i.l.r.s.c.DynamothDataCollector - Unable to invoke the method org.apache.commons.lang.math.NumberUtils.createDouble(java.lang.String) ["01l"]
com.sun.jdi.InvocationException: Exception occurred in target VM
	at com.sun.tools.jdi.InvokableTypeImpl.invokeMethod(InvokableTypeImpl.java:129) ~[tools.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:439) [nopol.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:1) [nopol.jar:na]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) [na:1.8.0_71]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_71]
23:45:55.115 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] org.apache.commons.lang.math.NumberUtils.createDouble((java.lang.String) numeric)
23:45:55.122 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] org.apache.commons.lang.StringUtils.isBlank((java.lang.String) str)
23:45:55.135 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] org.apache.commons.lang.StringUtils.isBlank((java.lang.String) mant)
23:45:55.139 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] org.apache.commons.lang.StringUtils.isBlank((java.lang.String) numeric)
23:45:55.144 [pool-174-thread-1] ERROR f.i.l.r.s.c.DynamothDataCollector - Unable to invoke the method org.apache.commons.lang.math.NumberUtils.createFloat(java.lang.String) ["01l"]
com.sun.jdi.InvocationException: Exception occurred in target VM
	at com.sun.tools.jdi.InvokableTypeImpl.invokeMethod(InvokableTypeImpl.java:129) ~[tools.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:439) [nopol.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:1) [nopol.jar:na]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) [na:1.8.0_71]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_71]
23:45:55.156 [pool-175-thread-1] ERROR f.i.l.r.s.c.DynamothDataCollector - Unable to invoke the method org.apache.commons.lang.math.NumberUtils.createFloat(java.lang.String) ["01l"]
com.sun.jdi.InvocationException: Exception occurred in target VM
	at com.sun.tools.jdi.InvokableTypeImpl.invokeMethod(InvokableTypeImpl.java:129) ~[tools.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:439) [nopol.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:1) [nopol.jar:na]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) [na:1.8.0_71]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_71]
23:45:55.159 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] org.apache.commons.lang.math.NumberUtils.createFloat((java.lang.String) numeric)
23:45:55.172 [pool-177-thread-1] ERROR f.i.l.r.s.c.DynamothDataCollector - Unable to invoke the method org.apache.commons.lang.math.NumberUtils.createBigDecimal(java.lang.String) ["01l"]
com.sun.jdi.InvocationException: Exception occurred in target VM
	at com.sun.tools.jdi.InvokableTypeImpl.invokeMethod(InvokableTypeImpl.java:129) ~[tools.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:439) [nopol.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:1) [nopol.jar:na]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) [na:1.8.0_71]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_71]
23:45:55.178 [pool-178-thread-1] ERROR f.i.l.r.s.c.DynamothDataCollector - Unable to invoke the method org.apache.commons.lang.math.NumberUtils.createBigDecimal(java.lang.String) ["01l"]
com.sun.jdi.InvocationException: Exception occurred in target VM
	at com.sun.tools.jdi.InvokableTypeImpl.invokeMethod(InvokableTypeImpl.java:129) ~[tools.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:439) [nopol.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:1) [nopol.jar:na]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) [na:1.8.0_71]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_71]
23:45:55.187 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] org.apache.commons.lang.math.NumberUtils.createBigDecimal((java.lang.String) numeric)
23:45:55.191 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] org.apache.commons.lang.math.NumberUtils.isDigits((java.lang.String) str)
23:45:55.198 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] org.apache.commons.lang.math.NumberUtils.isDigits((java.lang.String) mant)
23:45:55.206 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] org.apache.commons.lang.math.NumberUtils.isDigits((java.lang.String) numeric)
23:45:55.214 [pool-183-thread-1] ERROR f.i.l.r.s.c.DynamothDataCollector - Unable to invoke the method org.apache.commons.lang.math.NumberUtils.createBigInteger(java.lang.String) ["01l"]
com.sun.jdi.InvocationException: Exception occurred in target VM
	at com.sun.tools.jdi.InvokableTypeImpl.invokeMethod(InvokableTypeImpl.java:129) ~[tools.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:439) [nopol.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:1) [nopol.jar:na]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) [na:1.8.0_71]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_71]
23:45:55.218 [pool-184-thread-1] ERROR f.i.l.r.s.c.DynamothDataCollector - Unable to invoke the method org.apache.commons.lang.math.NumberUtils.createBigInteger(java.lang.String) ["01l"]
com.sun.jdi.InvocationException: Exception occurred in target VM
	at com.sun.tools.jdi.InvokableTypeImpl.invokeMethod(InvokableTypeImpl.java:129) ~[tools.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:439) [nopol.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:1) [nopol.jar:na]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) [na:1.8.0_71]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_71]
23:45:55.223 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] org.apache.commons.lang.math.NumberUtils.createBigInteger((java.lang.String) numeric)
23:45:55.235 [pool-186-thread-1] ERROR f.i.l.r.s.c.DynamothDataCollector - Unable to invoke the method org.apache.commons.lang.math.NumberUtils.createInteger(java.lang.String) ["01l"]
com.sun.jdi.InvocationException: Exception occurred in target VM
	at com.sun.tools.jdi.InvokableTypeImpl.invokeMethod(InvokableTypeImpl.java:129) ~[tools.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:439) [nopol.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:1) [nopol.jar:na]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) [na:1.8.0_71]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_71]
23:45:55.239 [pool-187-thread-1] ERROR f.i.l.r.s.c.DynamothDataCollector - Unable to invoke the method org.apache.commons.lang.math.NumberUtils.createInteger(java.lang.String) ["01l"]
com.sun.jdi.InvocationException: Exception occurred in target VM
	at com.sun.tools.jdi.InvokableTypeImpl.invokeMethod(InvokableTypeImpl.java:129) ~[tools.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:439) [nopol.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:1) [nopol.jar:na]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) [na:1.8.0_71]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_71]
23:45:55.250 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] org.apache.commons.lang.math.NumberUtils.createInteger((java.lang.String) numeric)
23:45:55.253 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] str.length()
23:45:55.257 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] mant.length()
23:45:55.261 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] numeric.length()
23:45:55.277 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] org.apache.commons.lang.math.NumberUtils.createBigDecimal((java.lang.String) numeric).intValue()
23:45:55.294 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] org.apache.commons.lang.math.NumberUtils.createBigInteger((java.lang.String) numeric).bitLength()
23:45:55.295 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] org.apache.commons.lang.math.NumberUtils.createBigInteger((java.lang.String) numeric).intValue()
23:45:55.296 [pool-7-thread-1] DEBUG f.i.l.r.s.DynamothCodeGenesisImpl - BreakpointEvent
23:45:55.297 [pool-7-thread-1] INFO  f.i.l.r.s.DynamothCodeGenesisImpl - [test] org.apache.commons.lang.math.NumberUtilsTest#testLang300 iteration 2
23:45:55.298 [pool-7-thread-1] DEBUG f.i.l.r.s.c.SpoonElementsCollector - [data] dec=null
23:45:55.298 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - Collect Level 1
23:45:55.298 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] str
23:45:55.298 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] lastChar
23:45:55.298 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] decPos
23:45:55.299 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] expPos
23:45:55.299 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] mant
23:45:55.299 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] dec
23:45:55.299 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] exp
23:45:55.299 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] numeric
23:45:55.299 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] allZeros
23:45:55.311 [pool-195-thread-1] ERROR f.i.l.r.s.c.DynamothDataCollector - Unable to invoke the method org.apache.commons.lang.math.NumberUtils.createLong(java.lang.String) ["1l"]
com.sun.jdi.InvocationException: Exception occurred in target VM
	at com.sun.tools.jdi.InvokableTypeImpl.invokeMethod(InvokableTypeImpl.java:129) ~[tools.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:439) [nopol.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:1) [nopol.jar:na]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) [na:1.8.0_71]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_71]
23:45:55.319 [pool-196-thread-1] ERROR f.i.l.r.s.c.DynamothDataCollector - Unable to invoke the method org.apache.commons.lang.math.NumberUtils.createLong(java.lang.String) ["1l"]
com.sun.jdi.InvocationException: Exception occurred in target VM
	at com.sun.tools.jdi.InvokableTypeImpl.invokeMethod(InvokableTypeImpl.java:129) ~[tools.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:439) [nopol.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:1) [nopol.jar:na]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) [na:1.8.0_71]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_71]
23:45:55.326 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] org.apache.commons.lang.math.NumberUtils.createLong((java.lang.String) numeric)
23:45:55.338 [pool-198-thread-1] ERROR f.i.l.r.s.c.DynamothDataCollector - Unable to invoke the method org.apache.commons.lang.math.NumberUtils.createDouble(java.lang.String) ["1l"]
com.sun.jdi.InvocationException: Exception occurred in target VM
	at com.sun.tools.jdi.InvokableTypeImpl.invokeMethod(InvokableTypeImpl.java:129) ~[tools.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:439) [nopol.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:1) [nopol.jar:na]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) [na:1.8.0_71]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_71]
23:45:55.342 [pool-199-thread-1] ERROR f.i.l.r.s.c.DynamothDataCollector - Unable to invoke the method org.apache.commons.lang.math.NumberUtils.createDouble(java.lang.String) ["1l"]
com.sun.jdi.InvocationException: Exception occurred in target VM
	at com.sun.tools.jdi.InvokableTypeImpl.invokeMethod(InvokableTypeImpl.java:129) ~[tools.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:439) [nopol.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:1) [nopol.jar:na]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) [na:1.8.0_71]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_71]
23:45:55.347 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] org.apache.commons.lang.math.NumberUtils.createDouble((java.lang.String) numeric)
23:45:55.362 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] org.apache.commons.lang.StringUtils.isBlank((java.lang.String) str)
23:45:55.370 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] org.apache.commons.lang.StringUtils.isBlank((java.lang.String) mant)
23:45:55.371 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] org.apache.commons.lang.StringUtils.isBlank((java.lang.String) numeric)
23:45:55.377 [pool-204-thread-1] ERROR f.i.l.r.s.c.DynamothDataCollector - Unable to invoke the method org.apache.commons.lang.math.NumberUtils.createFloat(java.lang.String) ["1l"]
com.sun.jdi.InvocationException: Exception occurred in target VM
	at com.sun.tools.jdi.InvokableTypeImpl.invokeMethod(InvokableTypeImpl.java:129) ~[tools.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:439) [nopol.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:1) [nopol.jar:na]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) [na:1.8.0_71]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_71]
23:45:55.394 [pool-205-thread-1] ERROR f.i.l.r.s.c.DynamothDataCollector - Unable to invoke the method org.apache.commons.lang.math.NumberUtils.createFloat(java.lang.String) ["1l"]
com.sun.jdi.InvocationException: Exception occurred in target VM
	at com.sun.tools.jdi.InvokableTypeImpl.invokeMethod(InvokableTypeImpl.java:129) ~[tools.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:439) [nopol.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:1) [nopol.jar:na]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) [na:1.8.0_71]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_71]
23:45:55.395 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] org.apache.commons.lang.math.NumberUtils.createFloat((java.lang.String) numeric)
23:45:55.401 [pool-207-thread-1] ERROR f.i.l.r.s.c.DynamothDataCollector - Unable to invoke the method org.apache.commons.lang.math.NumberUtils.createBigDecimal(java.lang.String) ["1l"]
com.sun.jdi.InvocationException: Exception occurred in target VM
	at com.sun.tools.jdi.InvokableTypeImpl.invokeMethod(InvokableTypeImpl.java:129) ~[tools.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:439) [nopol.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:1) [nopol.jar:na]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) [na:1.8.0_71]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_71]
23:45:55.409 [pool-208-thread-1] ERROR f.i.l.r.s.c.DynamothDataCollector - Unable to invoke the method org.apache.commons.lang.math.NumberUtils.createBigDecimal(java.lang.String) ["1l"]
com.sun.jdi.InvocationException: Exception occurred in target VM
	at com.sun.tools.jdi.InvokableTypeImpl.invokeMethod(InvokableTypeImpl.java:129) ~[tools.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:439) [nopol.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:1) [nopol.jar:na]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) [na:1.8.0_71]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_71]
23:45:55.421 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] org.apache.commons.lang.math.NumberUtils.createBigDecimal((java.lang.String) numeric)
23:45:55.425 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] org.apache.commons.lang.math.NumberUtils.isDigits((java.lang.String) str)
23:45:55.428 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] org.apache.commons.lang.math.NumberUtils.isDigits((java.lang.String) mant)
23:45:55.437 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] org.apache.commons.lang.math.NumberUtils.isDigits((java.lang.String) numeric)
23:45:55.441 [pool-213-thread-1] ERROR f.i.l.r.s.c.DynamothDataCollector - Unable to invoke the method org.apache.commons.lang.math.NumberUtils.createBigInteger(java.lang.String) ["1l"]
com.sun.jdi.InvocationException: Exception occurred in target VM
	at com.sun.tools.jdi.InvokableTypeImpl.invokeMethod(InvokableTypeImpl.java:129) ~[tools.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:439) [nopol.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:1) [nopol.jar:na]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) [na:1.8.0_71]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_71]
23:45:55.443 [pool-214-thread-1] ERROR f.i.l.r.s.c.DynamothDataCollector - Unable to invoke the method org.apache.commons.lang.math.NumberUtils.createBigInteger(java.lang.String) ["1l"]
com.sun.jdi.InvocationException: Exception occurred in target VM
	at com.sun.tools.jdi.InvokableTypeImpl.invokeMethod(InvokableTypeImpl.java:129) ~[tools.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:439) [nopol.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:1) [nopol.jar:na]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) [na:1.8.0_71]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_71]
23:45:55.466 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] org.apache.commons.lang.math.NumberUtils.createBigInteger((java.lang.String) numeric)
23:45:55.469 [pool-216-thread-1] ERROR f.i.l.r.s.c.DynamothDataCollector - Unable to invoke the method org.apache.commons.lang.math.NumberUtils.createInteger(java.lang.String) ["1l"]
com.sun.jdi.InvocationException: Exception occurred in target VM
	at com.sun.tools.jdi.InvokableTypeImpl.invokeMethod(InvokableTypeImpl.java:129) ~[tools.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:439) [nopol.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:1) [nopol.jar:na]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) [na:1.8.0_71]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_71]
23:45:55.478 [pool-217-thread-1] ERROR f.i.l.r.s.c.DynamothDataCollector - Unable to invoke the method org.apache.commons.lang.math.NumberUtils.createInteger(java.lang.String) ["1l"]
com.sun.jdi.InvocationException: Exception occurred in target VM
	at com.sun.tools.jdi.InvokableTypeImpl.invokeMethod(InvokableTypeImpl.java:129) ~[tools.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:439) [nopol.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:1) [nopol.jar:na]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) [na:1.8.0_71]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_71]
23:45:55.490 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] org.apache.commons.lang.math.NumberUtils.createInteger((java.lang.String) numeric)
23:45:55.503 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] str.length()
23:45:55.509 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] mant.length()
23:45:55.524 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] numeric.length()
23:45:55.534 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] org.apache.commons.lang.math.NumberUtils.createBigDecimal((java.lang.String) numeric).intValue()
23:45:55.541 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] org.apache.commons.lang.math.NumberUtils.createBigInteger((java.lang.String) numeric).bitLength()
23:45:55.547 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] org.apache.commons.lang.math.NumberUtils.createBigInteger((java.lang.String) numeric).intValue()
23:45:55.586 [pool-7-thread-1] DEBUG f.i.l.r.s.DynamothCodeGenesisImpl - BreakpointEvent
23:45:55.587 [pool-7-thread-1] INFO  f.i.l.r.s.DynamothCodeGenesisImpl - [test] org.apache.commons.lang.math.NumberUtilsTest#testIsNumber iteration 0
23:45:55.588 [pool-7-thread-1] DEBUG f.i.l.r.s.c.SpoonElementsCollector - [data] dec=null
23:45:55.588 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - Collect Level 1
23:45:55.588 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] str
23:45:55.588 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] lastChar
23:45:55.588 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] decPos
23:45:55.588 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] expPos
23:45:55.589 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] mant
23:45:55.589 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] dec
23:45:55.589 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] exp
23:45:55.589 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] numeric
23:45:55.589 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] allZeros
23:45:55.594 [pool-225-thread-1] ERROR f.i.l.r.s.c.DynamothDataCollector - Unable to invoke the method org.apache.commons.lang.math.NumberUtils.createLong(java.lang.String) ["22338L"]
com.sun.jdi.InvocationException: Exception occurred in target VM
	at com.sun.tools.jdi.InvokableTypeImpl.invokeMethod(InvokableTypeImpl.java:129) ~[tools.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:439) [nopol.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:1) [nopol.jar:na]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) [na:1.8.0_71]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_71]
23:45:55.597 [pool-226-thread-1] ERROR f.i.l.r.s.c.DynamothDataCollector - Unable to invoke the method org.apache.commons.lang.math.NumberUtils.createLong(java.lang.String) ["22338L"]
com.sun.jdi.InvocationException: Exception occurred in target VM
	at com.sun.tools.jdi.InvokableTypeImpl.invokeMethod(InvokableTypeImpl.java:129) ~[tools.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:439) [nopol.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:1) [nopol.jar:na]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) [na:1.8.0_71]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_71]
23:45:55.606 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] org.apache.commons.lang.math.NumberUtils.createLong((java.lang.String) numeric)
23:45:55.611 [pool-228-thread-1] ERROR f.i.l.r.s.c.DynamothDataCollector - Unable to invoke the method org.apache.commons.lang.math.NumberUtils.createDouble(java.lang.String) ["22338L"]
com.sun.jdi.InvocationException: Exception occurred in target VM
	at com.sun.tools.jdi.InvokableTypeImpl.invokeMethod(InvokableTypeImpl.java:129) ~[tools.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:439) [nopol.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:1) [nopol.jar:na]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) [na:1.8.0_71]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_71]
23:45:55.614 [pool-229-thread-1] ERROR f.i.l.r.s.c.DynamothDataCollector - Unable to invoke the method org.apache.commons.lang.math.NumberUtils.createDouble(java.lang.String) ["22338L"]
com.sun.jdi.InvocationException: Exception occurred in target VM
	at com.sun.tools.jdi.InvokableTypeImpl.invokeMethod(InvokableTypeImpl.java:129) ~[tools.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:439) [nopol.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:1) [nopol.jar:na]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) [na:1.8.0_71]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_71]
23:45:55.622 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] org.apache.commons.lang.math.NumberUtils.createDouble((java.lang.String) numeric)
23:45:55.629 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] org.apache.commons.lang.StringUtils.isBlank((java.lang.String) str)
23:45:55.632 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] org.apache.commons.lang.StringUtils.isBlank((java.lang.String) mant)
23:45:55.638 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] org.apache.commons.lang.StringUtils.isBlank((java.lang.String) numeric)
23:45:55.646 [pool-234-thread-1] ERROR f.i.l.r.s.c.DynamothDataCollector - Unable to invoke the method org.apache.commons.lang.math.NumberUtils.createFloat(java.lang.String) ["22338L"]
com.sun.jdi.InvocationException: Exception occurred in target VM
	at com.sun.tools.jdi.InvokableTypeImpl.invokeMethod(InvokableTypeImpl.java:129) ~[tools.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:439) [nopol.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:1) [nopol.jar:na]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) [na:1.8.0_71]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_71]
23:45:55.650 [pool-235-thread-1] ERROR f.i.l.r.s.c.DynamothDataCollector - Unable to invoke the method org.apache.commons.lang.math.NumberUtils.createFloat(java.lang.String) ["22338L"]
com.sun.jdi.InvocationException: Exception occurred in target VM
	at com.sun.tools.jdi.InvokableTypeImpl.invokeMethod(InvokableTypeImpl.java:129) ~[tools.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:439) [nopol.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:1) [nopol.jar:na]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) [na:1.8.0_71]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_71]
23:45:55.665 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] org.apache.commons.lang.math.NumberUtils.createFloat((java.lang.String) numeric)
23:45:55.670 [pool-237-thread-1] ERROR f.i.l.r.s.c.DynamothDataCollector - Unable to invoke the method org.apache.commons.lang.math.NumberUtils.createBigDecimal(java.lang.String) ["22338L"]
com.sun.jdi.InvocationException: Exception occurred in target VM
	at com.sun.tools.jdi.InvokableTypeImpl.invokeMethod(InvokableTypeImpl.java:129) ~[tools.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:439) [nopol.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:1) [nopol.jar:na]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) [na:1.8.0_71]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_71]
23:45:55.680 [pool-238-thread-1] ERROR f.i.l.r.s.c.DynamothDataCollector - Unable to invoke the method org.apache.commons.lang.math.NumberUtils.createBigDecimal(java.lang.String) ["22338L"]
com.sun.jdi.InvocationException: Exception occurred in target VM
	at com.sun.tools.jdi.InvokableTypeImpl.invokeMethod(InvokableTypeImpl.java:129) ~[tools.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:439) [nopol.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:1) [nopol.jar:na]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) [na:1.8.0_71]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_71]
23:45:55.698 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] org.apache.commons.lang.math.NumberUtils.createBigDecimal((java.lang.String) numeric)
23:45:55.706 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] org.apache.commons.lang.math.NumberUtils.isDigits((java.lang.String) str)
23:45:55.709 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] org.apache.commons.lang.math.NumberUtils.isDigits((java.lang.String) mant)
23:45:55.710 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] org.apache.commons.lang.math.NumberUtils.isDigits((java.lang.String) numeric)
23:45:55.730 [pool-243-thread-1] ERROR f.i.l.r.s.c.DynamothDataCollector - Unable to invoke the method org.apache.commons.lang.math.NumberUtils.createBigInteger(java.lang.String) ["22338L"]
com.sun.jdi.InvocationException: Exception occurred in target VM
	at com.sun.tools.jdi.InvokableTypeImpl.invokeMethod(InvokableTypeImpl.java:129) ~[tools.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:439) [nopol.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:1) [nopol.jar:na]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) [na:1.8.0_71]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_71]
23:45:55.732 [pool-244-thread-1] ERROR f.i.l.r.s.c.DynamothDataCollector - Unable to invoke the method org.apache.commons.lang.math.NumberUtils.createBigInteger(java.lang.String) ["22338L"]
com.sun.jdi.InvocationException: Exception occurred in target VM
	at com.sun.tools.jdi.InvokableTypeImpl.invokeMethod(InvokableTypeImpl.java:129) ~[tools.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:439) [nopol.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:1) [nopol.jar:na]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) [na:1.8.0_71]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_71]
23:45:55.733 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] org.apache.commons.lang.math.NumberUtils.createBigInteger((java.lang.String) numeric)
23:45:55.747 [pool-246-thread-1] ERROR f.i.l.r.s.c.DynamothDataCollector - Unable to invoke the method org.apache.commons.lang.math.NumberUtils.createInteger(java.lang.String) ["22338L"]
com.sun.jdi.InvocationException: Exception occurred in target VM
	at com.sun.tools.jdi.InvokableTypeImpl.invokeMethod(InvokableTypeImpl.java:129) ~[tools.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:439) [nopol.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:1) [nopol.jar:na]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) [na:1.8.0_71]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_71]
23:45:55.754 [pool-247-thread-1] ERROR f.i.l.r.s.c.DynamothDataCollector - Unable to invoke the method org.apache.commons.lang.math.NumberUtils.createInteger(java.lang.String) ["22338L"]
com.sun.jdi.InvocationException: Exception occurred in target VM
	at com.sun.tools.jdi.InvokableTypeImpl.invokeMethod(InvokableTypeImpl.java:129) ~[tools.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:439) [nopol.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:1) [nopol.jar:na]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) [na:1.8.0_71]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_71]
23:45:55.759 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] org.apache.commons.lang.math.NumberUtils.createInteger((java.lang.String) numeric)
23:45:55.765 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] str.length()
23:45:55.773 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] mant.length()
23:45:55.777 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] numeric.length()
23:45:55.789 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] org.apache.commons.lang.math.NumberUtils.createBigDecimal((java.lang.String) numeric).intValue()
23:45:55.797 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] org.apache.commons.lang.math.NumberUtils.createBigInteger((java.lang.String) numeric).bitLength()
23:45:55.803 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] org.apache.commons.lang.math.NumberUtils.createBigInteger((java.lang.String) numeric).intValue()
23:45:55.804 [pool-7-thread-1] DEBUG f.i.l.r.s.DynamothCodeGenesisImpl - BreakpointEvent
23:45:55.805 [pool-7-thread-1] INFO  f.i.l.r.s.DynamothCodeGenesisImpl - [test] org.apache.commons.lang.math.NumberUtilsTest#testIsNumber iteration 1
23:45:55.805 [pool-7-thread-1] DEBUG f.i.l.r.s.c.SpoonElementsCollector - [data] dec=null
23:45:55.805 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - Collect Level 1
23:45:55.806 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] str
23:45:55.806 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] lastChar
23:45:55.806 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] decPos
23:45:55.806 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] expPos
23:45:55.806 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] mant
23:45:55.806 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] dec
23:45:55.806 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] exp
23:45:55.806 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] numeric
23:45:55.806 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] allZeros
23:45:55.817 [pool-255-thread-1] ERROR f.i.l.r.s.c.DynamothDataCollector - Unable to invoke the method org.apache.commons.lang.math.NumberUtils.createLong(java.lang.String) ["1234E5l"]
com.sun.jdi.InvocationException: Exception occurred in target VM
	at com.sun.tools.jdi.InvokableTypeImpl.invokeMethod(InvokableTypeImpl.java:129) ~[tools.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:439) [nopol.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:1) [nopol.jar:na]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) [na:1.8.0_71]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_71]
23:45:55.827 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] org.apache.commons.lang.math.NumberUtils.createLong((java.lang.String) mant)
23:45:55.833 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] org.apache.commons.lang.math.NumberUtils.createLong((java.lang.String) exp)
23:45:55.837 [pool-258-thread-1] ERROR f.i.l.r.s.c.DynamothDataCollector - Unable to invoke the method org.apache.commons.lang.math.NumberUtils.createLong(java.lang.String) ["1234E5"]
com.sun.jdi.InvocationException: Exception occurred in target VM
	at com.sun.tools.jdi.InvokableTypeImpl.invokeMethod(InvokableTypeImpl.java:129) ~[tools.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:439) [nopol.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:1) [nopol.jar:na]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) [na:1.8.0_71]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_71]
23:45:55.845 [pool-259-thread-1] ERROR f.i.l.r.s.c.DynamothDataCollector - Unable to invoke the method org.apache.commons.lang.math.NumberUtils.createDouble(java.lang.String) ["1234E5l"]
com.sun.jdi.InvocationException: Exception occurred in target VM
	at com.sun.tools.jdi.InvokableTypeImpl.invokeMethod(InvokableTypeImpl.java:129) ~[tools.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:439) [nopol.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:1) [nopol.jar:na]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) [na:1.8.0_71]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_71]
23:45:55.857 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] org.apache.commons.lang.math.NumberUtils.createDouble((java.lang.String) mant)
23:45:55.866 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] org.apache.commons.lang.math.NumberUtils.createDouble((java.lang.String) exp)
23:45:55.868 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] org.apache.commons.lang.math.NumberUtils.createDouble((java.lang.String) numeric)
23:45:55.878 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] org.apache.commons.lang.StringUtils.isBlank((java.lang.String) str)
23:45:55.883 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] org.apache.commons.lang.StringUtils.isBlank((java.lang.String) mant)
23:45:55.886 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] org.apache.commons.lang.StringUtils.isBlank((java.lang.String) exp)
23:45:55.898 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] org.apache.commons.lang.StringUtils.isBlank((java.lang.String) numeric)
23:45:55.906 [pool-267-thread-1] ERROR f.i.l.r.s.c.DynamothDataCollector - Unable to invoke the method org.apache.commons.lang.math.NumberUtils.createFloat(java.lang.String) ["1234E5l"]
com.sun.jdi.InvocationException: Exception occurred in target VM
	at com.sun.tools.jdi.InvokableTypeImpl.invokeMethod(InvokableTypeImpl.java:129) ~[tools.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:439) [nopol.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:1) [nopol.jar:na]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) [na:1.8.0_71]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_71]
23:45:55.910 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] org.apache.commons.lang.math.NumberUtils.createFloat((java.lang.String) mant)
23:45:55.918 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] org.apache.commons.lang.math.NumberUtils.createFloat((java.lang.String) exp)
23:45:55.928 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] org.apache.commons.lang.math.NumberUtils.createFloat((java.lang.String) numeric)
23:45:55.939 [pool-271-thread-1] ERROR f.i.l.r.s.c.DynamothDataCollector - Unable to invoke the method org.apache.commons.lang.math.NumberUtils.createBigDecimal(java.lang.String) ["1234E5l"]
com.sun.jdi.InvocationException: Exception occurred in target VM
	at com.sun.tools.jdi.InvokableTypeImpl.invokeMethod(InvokableTypeImpl.java:129) ~[tools.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:439) [nopol.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:1) [nopol.jar:na]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) [na:1.8.0_71]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_71]
23:45:55.942 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] org.apache.commons.lang.math.NumberUtils.createBigDecimal((java.lang.String) mant)
23:45:55.946 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] org.apache.commons.lang.math.NumberUtils.createBigDecimal((java.lang.String) exp)
23:45:55.954 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] org.apache.commons.lang.math.NumberUtils.createBigDecimal((java.lang.String) numeric)
23:45:55.966 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] org.apache.commons.lang.math.NumberUtils.isDigits((java.lang.String) str)
23:45:55.974 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] org.apache.commons.lang.math.NumberUtils.isDigits((java.lang.String) mant)
23:45:55.975 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] org.apache.commons.lang.math.NumberUtils.isDigits((java.lang.String) exp)
23:45:55.978 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] org.apache.commons.lang.math.NumberUtils.isDigits((java.lang.String) numeric)
23:45:55.986 [pool-279-thread-1] ERROR f.i.l.r.s.c.DynamothDataCollector - Unable to invoke the method org.apache.commons.lang.math.NumberUtils.createBigInteger(java.lang.String) ["1234E5l"]
com.sun.jdi.InvocationException: Exception occurred in target VM
	at com.sun.tools.jdi.InvokableTypeImpl.invokeMethod(InvokableTypeImpl.java:129) ~[tools.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:439) [nopol.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:1) [nopol.jar:na]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) [na:1.8.0_71]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_71]
23:45:55.999 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] org.apache.commons.lang.math.NumberUtils.createBigInteger((java.lang.String) mant)
23:45:56.002 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] org.apache.commons.lang.math.NumberUtils.createBigInteger((java.lang.String) exp)
23:45:56.005 [pool-282-thread-1] ERROR f.i.l.r.s.c.DynamothDataCollector - Unable to invoke the method org.apache.commons.lang.math.NumberUtils.createBigInteger(java.lang.String) ["1234E5"]
com.sun.jdi.InvocationException: Exception occurred in target VM
	at com.sun.tools.jdi.InvokableTypeImpl.invokeMethod(InvokableTypeImpl.java:129) ~[tools.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:439) [nopol.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:1) [nopol.jar:na]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) [na:1.8.0_71]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_71]
23:45:56.010 [pool-283-thread-1] ERROR f.i.l.r.s.c.DynamothDataCollector - Unable to invoke the method org.apache.commons.lang.math.NumberUtils.createInteger(java.lang.String) ["1234E5l"]
com.sun.jdi.InvocationException: Exception occurred in target VM
	at com.sun.tools.jdi.InvokableTypeImpl.invokeMethod(InvokableTypeImpl.java:129) ~[tools.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:439) [nopol.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:1) [nopol.jar:na]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) [na:1.8.0_71]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_71]
23:45:56.026 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] org.apache.commons.lang.math.NumberUtils.createInteger((java.lang.String) mant)
23:45:56.030 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] org.apache.commons.lang.math.NumberUtils.createInteger((java.lang.String) exp)
23:45:56.031 [pool-286-thread-1] ERROR f.i.l.r.s.c.DynamothDataCollector - Unable to invoke the method org.apache.commons.lang.math.NumberUtils.createInteger(java.lang.String) ["1234E5"]
com.sun.jdi.InvocationException: Exception occurred in target VM
	at com.sun.tools.jdi.InvokableTypeImpl.invokeMethod(InvokableTypeImpl.java:129) ~[tools.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:439) [nopol.jar:na]
	at fr.inria.lille.repair.synthesis.collect.DynamothDataCollector$2.call(DynamothDataCollector.java:1) [nopol.jar:na]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) [na:1.8.0_71]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) [na:1.8.0_71]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_71]
23:45:56.035 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] str.length()
23:45:56.043 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] mant.length()
23:45:56.049 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] exp.length()
23:45:56.054 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] numeric.length()
23:45:56.069 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] org.apache.commons.lang.math.NumberUtils.createBigDecimal((java.lang.String) mant).intValue()
23:45:56.086 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] org.apache.commons.lang.math.NumberUtils.createBigDecimal((java.lang.String) exp).intValue()
23:45:56.099 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] org.apache.commons.lang.math.NumberUtils.createBigDecimal((java.lang.String) numeric).intValue()
23:45:56.105 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] org.apache.commons.lang.math.NumberUtils.createBigInteger((java.lang.String) mant).bitLength()
23:45:56.112 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] org.apache.commons.lang.math.NumberUtils.createBigInteger((java.lang.String) mant).intValue()
23:45:56.117 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] org.apache.commons.lang.math.NumberUtils.createBigInteger((java.lang.String) exp).bitLength()
23:45:56.121 [pool-7-thread-1] DEBUG f.i.l.r.s.c.DynamothDataCollector - [data] org.apache.commons.lang.math.NumberUtils.createBigInteger((java.lang.String) exp).intValue()
23:45:56.124 [pool-7-thread-1] DEBUG f.i.l.r.s.DynamothCodeGenesisImpl - Exit
23:45:56.143 [pool-7-thread-1] DEBUG f.i.l.r.s.collect.DataCombiner - [combine] start on 30 elements
23:45:56.189 [pool-7-thread-1] DEBUG f.i.l.r.s.collect.DataCombiner - [combine] end 128 evaluated elements


========= Info ==========
Nb constants             10
Nb method invocations    14
Nb field access          0
Nb variables             6
Total                    30
Nb evaluated expressions 32
Init Execution time      2770 ms
Collect Execution time   2693 ms
Combine Execution time   65.0 ms
Nb Combination par sec   492 combinations/sec
Total Execution time     5529 ms
Nb line execution        9
Nb results               1

Results:
1. exp == dec


========================
         Stat
========================
Operators: 
AND 12
EQ 10
PLUS 9
OR 6
MINUS 5
GT 5
LT 3
Methods: 
length() 6
substring(int,int) 6
isAllZeros(java.lang.String) 4
startsWith(java.lang.String) 3
charAt(int) 3
indexOf(int) 3
createLong(java.lang.String) 2
createDouble(java.lang.String) 2
floatValue() 2
substring(int) 2
createFloat(java.lang.String) 2
createBigDecimal(java.lang.String) 2
isInfinite() 2
isInfinite() 2
createBigInteger(java.lang.String) 2
createInteger(java.lang.String) 2
isDigit(char) 2
floatValue() 1
doubleValue() 1
isBlank(java.lang.String) 1
isDigits(java.lang.String) 1
Variable: 
str 33
expPos 11
numeric 8
exp 8
d 6
f 6
decPos 5
dec 5
mant 5
allZeros 4
lastChar 2

 & 10 & 14 & 0 & 6 & 30 & 32 & 5530 ms & 9 &
number of passing is 0
number of failing is 0
23:45:56.201 [pool-7-thread-1] INFO  f.i.l.repair.nopol.patch.TestPatch - Applying patch: org.apache.commons.lang.math.NumberUtils:452: CONDITIONAL exp == dec
23:45:56.948 [pool-7-thread-1] INFO  f.i.l.repair.nopol.patch.TestPatch - Running test suite to check the patch "exp == dec" is working
23:45:57.007 [pool-2-thread-1] INFO  fr.inria.lille.repair.nopol.NoPol - ----INFORMATION----
23:45:57.261 [pool-2-thread-1] INFO  fr.inria.lille.repair.nopol.NoPol - Nb classes : 77
23:45:57.261 [pool-2-thread-1] INFO  fr.inria.lille.repair.nopol.NoPol - Nb methods : 1700
23:45:57.261 [pool-2-thread-1] INFO  fr.inria.lille.repair.nopol.NoPol - Nb Statements Analyzed : 2
23:45:57.264 [pool-2-thread-1] INFO  fr.inria.lille.repair.nopol.NoPol - Nb Statements with Angelic Value Found : 1
23:45:57.264 [pool-2-thread-1] INFO  fr.inria.lille.repair.nopol.NoPol - NoPol Execution time : 48431ms
23:45:57.264 [pool-2-thread-1] INFO  fr.inria.lille.repair.nopol.NoPol - 
23:45:57.264 [pool-2-thread-1] INFO  fr.inria.lille.repair.nopol.NoPol - ----STOP_BY_PATCH_FOUND----
23:45:57.264 [pool-2-thread-1] INFO  fr.inria.lille.repair.nopol.NoPol - Number of right compilations is  1
23:45:57.264 [pool-2-thread-1] INFO  fr.inria.lille.repair.nopol.NoPol - Number of failing compilations is  0
23:45:57.264 [pool-2-thread-1] INFO  fr.inria.lille.repair.nopol.NoPol - exp == dec
23:45:57.300 [pool-2-thread-1] INFO  fr.inria.lille.repair.nopol.NoPol - Nb test that executes the patch: 3
23:45:57.300 [pool-2-thread-1] INFO  fr.inria.lille.repair.nopol.NoPol - org.apache.commons.lang.math.NumberUtils:452: CONDITIONAL
23:45:57.403 [pool-2-thread-1] INFO  fr.inria.lille.repair.nopol.NoPol - --- a/src/java/org/apache/commons/lang/math/NumberUtils.java
+++ b/src/java/org/apache/commons/lang/math/NumberUtils.java
@@ -451,6 +451,3 @@
                 case 'L' :
-                    if (dec == null
-                        && exp == null
-                        && isDigits(numeric.substring(1))
-                        && (numeric.charAt(0) == '-' || Character.isDigit(numeric.charAt(0)))) {
+                    if (exp == dec) {
                         try {
@@ -461,3 +458,3 @@
                         return createBigInteger(numeric);
-
+                        
                     }

PATCH
23.64user 0.88system 0:50.41elapsed 48%CPU (0avgtext+0avgdata 539088maxresident)k
8inputs+5840outputs (0major+168251minor)pagefaults 0swaps


Node: ubuntu



Date: Sun Jul 28 23:45:57 EDT 2019

